<!DOCTYPE html>
<html>
<head>
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta charset="UTF-8">
<title>WebGestalt (WEB-based GEne SeT AnaLysis Toolkit)</title>
<link rel="icon" type="image/png" href="https://s3-us-west-2.amazonaws.com/webgestalt/favicon.png">

<script src="https://polyfill.io/v3/polyfill.min.js?flags=gated&features=es2015%2Ces2016%2Cfetch"></script>
<script src="https://2024.webgestalt.org/assets/1.0/build/vendor.min.js"></script>
	<script>window.cytoscape = null;</script>
	<link rel="stylesheet" href="https://2024.webgestalt.org/assets/1.0/build/vendor.css">
	<link rel="stylesheet" href="https://2024.webgestalt.org/assets/1.0/build/wg.css">
<link rel="stylesheet" href="https://cdn.materialdesignicons.com/2.5.94/css/materialdesignicons.min.css">
</head>
<body>
	<script type="text/javascript" src="https://www.termsfeed.com/public/cookie-consent/4.1.0/cookie-consent.js" charset="UTF-8"></script>
<script type="text/javascript" charset="UTF-8">
document.addEventListener('DOMContentLoaded', function () {
cookieconsent.run({"notice_banner_type":"simple","consent_type":"express","palette":"light","language":"en","page_load_consent_levels":["strictly-necessary"],"notice_banner_reject_button_hide":false,"preferences_center_close_button_hide":false,"page_refresh_confirmation_buttons":false});
});
</script>

<!-- Google Analytics -->
<script type="text/plain" data-cookie-consent="tracking" async src="https://www.googletagmanager.com/gtag/js?id=G-THG9Q3WPEJ"></script>
<script type="text/plain" data-cookie-consent="tracking">
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-THG9Q3WPEJ');
</script>
<!-- end of Google Analytics-->
	
	<noscript>Free cookie consent management tool by <a href="https://www.termsfeed.com/">TermsFeed</a></noscript>
	<!-- End Cookie Consent by TermsFeed https://www.TermsFeed.com -->
<header>
	<div id="header-img"><a href="https://2024.webgestalt.org"><img border="0" src="https://2024.webgestalt.org/images/gsa.png"></a></div>
	<div id="header-title">
		<p id="header-title-text">WEB-based GEne SeT AnaLysis Toolkit</p>
		<p id="header-subtitle-text">Translating gene lists into biological insights...</p>
	</div>
</header>
<hr><main>
<div id="app">
<div class="loading-preinit" v-cloak><div class="loading-icon"></div></div>
<div class="wrapper" v-cloak>
<div class="level">
	<div class="level-left">
		<div class="level-item">
			<h4 class="title is-4 has-text-primary">Summary</h4>
		</div>
	</div>
	<div class="level-right">
		<div class="level-item">
			<a class="button is-text" href="Project_wg_result1720045990.zip" target="_blank">Result Download</a>
		</div>
	</div>
</div>

<section id="summary" class="summary">
<div class="card" v-bind:class="{active: summaryShown}" >
	<div class="card-header" v-on:click="summaryShown=!summaryShown">
		<p class="card-header-title">Job summary</p>
		<a href="#" class="card-header-icon">
			<b-icon :icon="summaryShown ? 'chevron-up' : 'chevron-down'"></b-icon>
		</a>
	</div>
	<div class="card-content content" v-show="summaryShown">
		<ul id="summary-list"> 
		<li><b>Enrichment method:</b> ORA</li>
		<li><b>Organism: </b>hsapiens</li>
		<li>
				<b>Enrichment Categories: </b>pathway_WikiPathways
				<a class="has-text-info" href="https://2024.webgestalt.org/api/geneset?organism=hsapiens&database=pathway_WikiPathways&fileType=gmt">
					<b-icon icon="download" size="is-small"></b-icon>
				</a>
		</li>

			<li><b>Interesting list: </b>textAreaUpload_1720045990.txt. <b>ID type: </b>genesymbol</li>
			<li>The interesting list contains <b>22</b> user IDs in which <b>20</b> user IDs are unambiguously mapped to <b>20</b> unique entrezgene IDs and <b>2</b> user IDs can not be mapped to any entrezgene ID.</li>
			<li>The GO Slim summary are based upon the <b>20</b> unique entrezgene IDs.</li>
		<li>Among <b>20</b> unique entrezgene IDs, <b>14</b> IDs are annotated to the selected functional categories and also in the reference list, which are used for the enrichment analysis.</li>
		<li><b>Reference list: </b> all mapped entrezgene IDs from the selected platform genome
			<a class="has-text-info" href="https://2024.webgestalt.org/api/reference?organism=hsapiens&referenceSet=genome">
				<b-icon icon="download" size="is-small"></b-icon>
			</a>
		</li>
	<li>The reference list can be mapped to <b>193394</b> entrezgene IDs and <b> 8382</b> IDs are annotated to the selected functional categories that are used as the reference for the enrichment analysis.</li>
</ul> 

<p><b>Parameters for the enrichment analysis:</b></p>
<ul><li><b>Minimum number of IDs in the category: </b>5</li>
<li><b>Maximum number of IDs in the category: </b>2000</li>
<li><b>FDR Method: </b>BH</li>
<li><b>Significance Level:</b> Top 10</li>
</ul>
	<p>Based on the above parameters, <b>10</b> categories are identified as enriched categories and all are shown in this report.</p>

	</div>
</div>
</section>
<section id="go-summary" class="summary">
<div class="card" v-bind:class="{active: goShown}" >
	<div class="card-header" v-on:click="goShown=!goShown">
		<p class="card-header-title">GO Slim summary for the user uploaded IDs</p>
		<a href="#" class="card-header-icon">
			<b-icon :icon="goShown ? 'chevron-up' : 'chevron-down'"></b-icon>
		</a>
	</div>
	<div v-show="goShown">
		<div class="card-content">
			<p>Each Biological Process, Cellular Component and Molecular Function category is represented by a red, blue and green bar, repectively.</p>
			<p>The height of the bar represents the number of IDs in the user list and also in the category.</p>
		</div>
		<div class="card-image">
			<img src="goslim_summary_wg_result1720045990.png" width="100%" height="100%" alt="Bar charts of enriched GO Slim terms"/>
		</div>
	</div>
</div>
</section> <h4 class="title is-4 has-text-primary">Enrichment Results</h4>
<section id="enrich-result">
	<p class="select-redundancy">
		<b-tooltip label="Enriched analyte sets are post-processed by two methods to reduce redundancy." type="is-dark" animated multilined dashed>
			Redundancy reduction:
		</b-tooltip>
		<b-radio v-model="redundancy" native-value="all">None</b-radio>
		<b-radio v-model="redundancy" native-value="wsc">
			<b-tooltip label="Find top analyte sets while maximizing analyte coverage." type="is-dark" animated multilined dashed>
				Weighted set cover
			</b-tooltip>
		</b-radio>
	</p>
	<b-tabs v-model="activeTab" type="is-toggle">
		<b-tab-item label="Table">
			<wg-result-table
				:data="enrichmentData"
				:method="enrichMethod"
			></wg-result-table>
		</b-tab-item>
		<b-tab-item label="Bar chart">
		<div class="vertical-center-container">
			<wg-bar-plot
				:dimensions="bar.dimensions"
				:label-width="bar.labelWidth"
				:data="enrichmentData"
				:value-key="bar.valueKey"
				:x-label="bar.xLabel">
			</wg-bar-plot>
		</div>
		</b-tab-item>
		<b-tab-item label="Volcano plot">
		<div class="vertical-center-container">
			<wg-volcano-plot
				:data="volcanoData"
				:x="volcano.x"
				:y="volcano.y"
				:dimensions="volcano.dimensions"
				:x-label="volcano.xLabel"
				:label-types="volcano.labelTypes"
				:x-transform="volcano.xTransform"
				:radius="volcano.radius"
				:color="volcano.color"
				:tooltip-content="volcano.tooltipContent"
				:show-all-labels="false"
				:auto-place-labels="true"
			></wg-volcano-plot>
		</div>
		</b-tab-item>
	</b-tabs>
	<wg-gene-table
		:data="geneTables"
		:info="enrichment"
		:standard-id="standardId"
		:method="enrichMethod"
		:mapped-input-size="mappedInputSize"
		:organism="organism"
		:host-name="hostName"
		:sets-with-net="false"
	></wg-gene-table>
</section> 

</div>
</div>
</main>
	<script type="text/javascript" src="https://2024.webgestalt.org/assets/1.0/build/wg.min.js"></script>
<script type="text/javascript">
var enrichment = [{"geneSet":"WP5213","description":"Amino acid metabolism in triple negative breast cancer cells","link":"http://www.wikipathways.org/wpi/PathwayWidget.php?id=WP5213&label[]=PSAT1&colors=steelblue","size":7,"overlap":1,"expect":0.011691720353137676,"enrichmentRatio":85.530612244897952,"pValue":0.011637443981543162,"FDR":1,"overlapId":"29968"},{"geneSet":"WP4688","description":"Serine metabolism","link":"http://www.wikipathways.org/wpi/PathwayWidget.php?id=WP4688&label[]=PSAT1&colors=steelblue","size":8,"overlap":1,"expect":0.01336196611787163,"enrichmentRatio":74.839285714285708,"pValue":0.013289632135121911,"FDR":1,"overlapId":"29968"},{"geneSet":"WP4891","description":"COVID 19 adverse outcome pathway","link":"http://www.wikipathways.org/wpi/PathwayWidget.php?id=WP4891&label[]=CXCL8&colors=steelblue","size":15,"overlap":1,"expect":0.025053686471009307,"enrichmentRatio":39.914285714285711,"pValue":0.024783337613087139,"FDR":1,"overlapId":"3576"},{"geneSet":"WP3664","description":"Regulation of Wnt B catenin signaling by small molecule compounds","link":"http://www.wikipathways.org/wpi/PathwayWidget.php?id=WP3664&label[]=SFRP4&colors=steelblue","size":17,"overlap":1,"expect":0.028394178000477215,"enrichmentRatio":35.218487394957982,"pValue":0.028044345260017835,"FDR":1,"overlapId":"6424"},{"geneSet":"WP4493","description":"Cells and molecules involved in local acute inflammatory response","link":"http://www.wikipathways.org/wpi/PathwayWidget.php?id=WP4493&label[]=CXCL8&colors=steelblue","size":17,"overlap":1,"expect":0.028394178000477215,"enrichmentRatio":35.218487394957982,"pValue":0.028044345260017835,"FDR":1,"overlapId":"3576"},{"geneSet":"WP5172","description":"Iron metabolism disorders","link":"http://www.wikipathways.org/wpi/PathwayWidget.php?id=WP5172&label[]=CP&colors=steelblue","size":17,"overlap":1,"expect":0.028394178000477215,"enrichmentRatio":35.218487394957982,"pValue":0.028044345260017835,"FDR":1,"overlapId":"1356"},{"geneSet":"WP3287","description":"Overview of nanoparticle effects","link":"http://www.wikipathways.org/wpi/PathwayWidget.php?id=WP3287&label[]=CXCL8&colors=steelblue","size":19,"overlap":1,"expect":0.031734669529945123,"enrichmentRatio":31.511278195488721,"pValue":0.031295224955538652,"FDR":1,"overlapId":"3576"},{"geneSet":"WP4478","description":"LTF danger signal response pathway","link":"http://www.wikipathways.org/wpi/PathwayWidget.php?id=WP4478&label[]=CXCL8&colors=steelblue","size":19,"overlap":1,"expect":0.031734669529945123,"enrichmentRatio":31.511278195488721,"pValue":0.031295224955538652,"FDR":1,"overlapId":"3576"},{"geneSet":"WP4884","description":"Pathogenesis of SARS CoV 2 mediated by nsp9 nsp10 complex","link":"http://www.wikipathways.org/wpi/PathwayWidget.php?id=WP4884&label[]=CXCL8&colors=steelblue","size":21,"overlap":1,"expect":0.035075161059413031,"enrichmentRatio":28.510204081632651,"pValue":0.034536005742294755,"FDR":1,"overlapId":"3576"},{"geneSet":"WP262","description":"Ebstein Barr virus LMP1 signaling","link":"http://www.wikipathways.org/wpi/PathwayWidget.php?id=WP262&label[]=CXCL8&colors=steelblue","size":23,"overlap":1,"expect":0.038415652588880939,"enrichmentRatio":26.031055900621116,"pValue":0.03776671658663959,"FDR":1,"overlapId":"3576"}];
var background = [{"geneSet":"WP3613","enrichmentRatio":22.174603174603178,"FDR":1,"overlap":1},{"geneSet":"WP3851","enrichmentRatio":21.382653061224488,"FDR":1,"overlap":1},{"geneSet":"WP5115","enrichmentRatio":5.4927916120576672,"FDR":1,"overlap":2},{"geneSet":"WP5113","enrichmentRatio":18.709821428571427,"FDR":1,"overlap":1},{"geneSet":"WP5312","enrichmentRatio":18.709821428571427,"FDR":1,"overlap":1},{"geneSet":"WP4481","enrichmentRatio":18.142857142857142,"FDR":1,"overlap":1},{"geneSet":"WP5088","enrichmentRatio":18.142857142857142,"FDR":1,"overlap":1},{"geneSet":"WP3617","enrichmentRatio":17.10612244897959,"FDR":1,"overlap":1},{"geneSet":"WP5285","enrichmentRatio":15.351648351648352,"FDR":1,"overlap":1},{"geneSet":"WP2828","enrichmentRatio":14.967857142857143,"FDR":1,"overlap":1},{"geneSet":"WP500","enrichmentRatio":14.967857142857143,"FDR":1,"overlap":1},{"geneSet":"WP4290","enrichmentRatio":14.255102040816325,"FDR":1,"overlap":1},{"geneSet":"WP716","enrichmentRatio":13.923588039867111,"FDR":1,"overlap":1},{"geneSet":"WP4577","enrichmentRatio":13.607142857142858,"FDR":1,"overlap":1},{"geneSet":"WP5078","enrichmentRatio":13.015527950310558,"FDR":1,"overlap":1},{"geneSet":"WP5347","enrichmentRatio":12.473214285714286,"FDR":1,"overlap":1},{"geneSet":"WP286","enrichmentRatio":11.974285714285715,"FDR":1,"overlap":1},{"geneSet":"WP3646","enrichmentRatio":10.691326530612244,"FDR":1,"overlap":1},{"geneSet":"WP558","enrichmentRatio":10.322660098522167,"FDR":1,"overlap":1},{"geneSet":"WP3865","enrichmentRatio":9.9785714285714278,"FDR":1,"overlap":1},{"geneSet":"WP3624","enrichmentRatio":9.3549107142857135,"FDR":1,"overlap":1},{"geneSet":"WP4329","enrichmentRatio":9.3549107142857135,"FDR":1,"overlap":1},{"geneSet":"WP4313","enrichmentRatio":9.210989010989012,"FDR":1,"overlap":1},{"geneSet":"WP5039","enrichmentRatio":9.0714285714285712,"FDR":1,"overlap":1},{"geneSet":"WP2525","enrichmentRatio":8.8046218487394956,"FDR":1,"overlap":1},{"geneSet":"WP2880","enrichmentRatio":8.5530612244897952,"FDR":1,"overlap":1},{"geneSet":"WP4341","enrichmentRatio":7.9828571428571422,"FDR":1,"overlap":1},{"geneSet":"WP5055","enrichmentRatio":7.9828571428571422,"FDR":1,"overlap":1},{"geneSet":"WP5353","enrichmentRatio":7.3013937282229966,"FDR":1,"overlap":1},{"geneSet":"WP4018","enrichmentRatio":7.0436974789915974,"FDR":1,"overlap":1},{"geneSet":"WP5094","enrichmentRatio":6.8035714285714288,"FDR":1,"overlap":1},{"geneSet":"WP4336","enrichmentRatio":6.7271268057784912,"FDR":1,"overlap":1},{"geneSet":"WP2328","enrichmentRatio":6.6523809523809518,"FDR":1,"overlap":1},{"geneSet":"WP35","enrichmentRatio":6.5792778649921502,"FDR":1,"overlap":1},{"geneSet":"WP3925","enrichmentRatio":6.5792778649921502,"FDR":1,"overlap":1},{"geneSet":"WP5355","enrichmentRatio":6.5077639751552789,"FDR":1,"overlap":1},{"geneSet":"WP2355","enrichmentRatio":6.4377880184331806,"FDR":1,"overlap":1},{"geneSet":"WP2806","enrichmentRatio":6.3022556390977442,"FDR":1,"overlap":1},{"geneSet":"WP4258","enrichmentRatio":6.1093294460641401,"FDR":1,"overlap":1},{"geneSet":"WP75","enrichmentRatio":5.8127600554785017,"FDR":1,"overlap":1},{"geneSet":"WP5090","enrichmentRatio":5.595460614152203,"FDR":1,"overlap":1},{"geneSet":"WP428","enrichmentRatio":5.2518796992481196,"FDR":1,"overlap":1},{"geneSet":"WP4659","enrichmentRatio":5.2062111801242237,"FDR":1,"overlap":1},{"geneSet":"WP2431","enrichmentRatio":4.9892857142857139,"FDR":1,"overlap":1},{"geneSet":"WP5095","enrichmentRatio":4.6411960132890364,"FDR":1,"overlap":1},{"geneSet":"WP236","enrichmentRatio":4.5703380588876774,"FDR":1,"overlap":1},{"geneSet":"WP5083","enrichmentRatio":4.2765306122448976,"FDR":1,"overlap":1},{"geneSet":"WP4673","enrichmentRatio":4.1290640394088669,"FDR":1,"overlap":1},{"geneSet":"WP5224","enrichmentRatio":4.1290640394088669,"FDR":1,"overlap":1},{"geneSet":"WP4666","enrichmentRatio":3.9649952696310309,"FDR":1,"overlap":1},{"geneSet":"WP4396","enrichmentRatio":3.8877551020408161,"FDR":1,"overlap":1},{"geneSet":"WP4262","enrichmentRatio":3.8626728110599085,"FDR":1,"overlap":1},{"geneSet":"WP481","enrichmentRatio":3.7419642857142859,"FDR":1,"overlap":1},{"geneSet":"WP706","enrichmentRatio":3.7419642857142859,"FDR":1,"overlap":1},{"geneSet":"WP2877","enrichmentRatio":3.2016806722689077,"FDR":1,"overlap":1},{"geneSet":"WP5124","enrichmentRatio":2.2764801738185767,"FDR":1,"overlap":1},{"geneSet":"WP2059","enrichmentRatio":2.2340085287846483,"FDR":1,"overlap":1},{"geneSet":"WP2882","enrichmentRatio":1.9067333939945406,"FDR":1,"overlap":1},{"geneSet":"WP3888","enrichmentRatio":1.3859126984126986,"FDR":1,"overlap":1},{"geneSet":"WP5087","enrichmentRatio":1.3638138626749106,"FDR":1,"overlap":1}];
var geneTables = {"WP5213":[{"userId":"PSAT1","geneSymbol":"PSAT1","geneName":"phosphoserine aminotransferase 1","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=29968","entrezgene":"29968"}],"WP4688":[{"userId":"PSAT1","geneSymbol":"PSAT1","geneName":"phosphoserine aminotransferase 1","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=29968","entrezgene":"29968"}],"WP4891":[{"userId":"CXCL8","geneSymbol":"CXCL8","geneName":"C-X-C motif chemokine ligand 8","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=3576","entrezgene":"3576"}],"WP3664":[{"userId":"SFRP4","geneSymbol":"SFRP4","geneName":"secreted frizzled related protein 4","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=6424","entrezgene":"6424"}],"WP4493":[{"userId":"CXCL8","geneSymbol":"CXCL8","geneName":"C-X-C motif chemokine ligand 8","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=3576","entrezgene":"3576"}],"WP5172":[{"userId":"CP","geneSymbol":"CP","geneName":"ceruloplasmin","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=1356","entrezgene":"1356"}],"WP3287":[{"userId":"CXCL8","geneSymbol":"CXCL8","geneName":"C-X-C motif chemokine ligand 8","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=3576","entrezgene":"3576"}],"WP4478":[{"userId":"CXCL8","geneSymbol":"CXCL8","geneName":"C-X-C motif chemokine ligand 8","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=3576","entrezgene":"3576"}],"WP4884":[{"userId":"CXCL8","geneSymbol":"CXCL8","geneName":"C-X-C motif chemokine ligand 8","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=3576","entrezgene":"3576"}],"WP262":[{"userId":"CXCL8","geneSymbol":"CXCL8","geneName":"C-X-C motif chemokine ligand 8","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=3576","entrezgene":"3576"}]};
var clusters = {"wsc":{"representatives":["WP5213","WP4891","WP3664","WP5172"],"coverage":[1]}};

wgLib({
	enrichDatabase: "pathway_WikiPathways",
	organism: "hsapiens",
	hostName: "https://2024.webgestalt.org",
	enrichment: enrichment,
	background: background,
	geneTables: geneTables,
	clusters: clusters,
	enrichMethod: "ora",
	mappedInputSize: 14,
	standardId: "entrezgene",
	volcano: {
		labelTypes: ["geneSet", "description"],
		dimensions: { width: 960, height: 500 },
		autoPlaceLabels: true,
		y: "FDR",

		x: "enrichmentRatio",
		xLabel: "log2 of enrichment ratio",
		radius: "overlap",
		color: "overlap",
			tooltipContent: ["geneSet", "description", "enrichmentRatio", "FDR", "pValue", "overlap"]
	},
	bar: {
		dimensions: { width: 960, height: 300 },
		labelWidth: 200,
		tooltipContent: ["geneSet", "description", "enrichmentRatio", "FDR", "pValue", "overlap"],
		valueKey: "enrichmentRatio",
		xLabel: "Enrichment ratio"
	}
})


</script>
<footer class="footer has-text-centered">
<p>WebGestalt is currently developed and maintained by Yuxing Liao, Suhas Vasaikar, Zhiao Shi and Bing Zhang at the <a target="_new" href=http://www.zhang-lab.org> Zhang Lab</a>. Other people who have made significant contribution to the project include Jing Wang, Dexter Duncan, Stefan Kirov and Jay Snoddy.</p>
<p><b>Funding credits:</b> NIH/NCI (U24 CA210954); Leidos (15X038); CPRIT (RR160027); NIH/NIAAA (U01 AA016662, U01 AA013512); NIH/NIDA (P01 DA015027); NIH/NIMH (P50 MH078028, P50 MH096972); NIH/NCI (U24 CA159988); NIH/NIGMS (R01 GM088822).
</p>
</footer></body>
</html>