<!DOCTYPE html>
<html>
<head>
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta charset="UTF-8">
<title>WebGestalt (WEB-based GEne SeT AnaLysis Toolkit)</title>
<link rel="icon" type="image/png" href="https://s3-us-west-2.amazonaws.com/webgestalt/favicon.png">

<script src="https://polyfill.io/v3/polyfill.min.js?flags=gated&features=es2015%2Ces2016%2Cfetch"></script>
<script src="https://2024.webgestalt.org/assets/1.0/build/vendor.min.js"></script>
	<script>window.cytoscape = null;</script>
	<link rel="stylesheet" href="https://2024.webgestalt.org/assets/1.0/build/vendor.css">
	<link rel="stylesheet" href="https://2024.webgestalt.org/assets/1.0/build/wg.css">
<link rel="stylesheet" href="https://cdn.materialdesignicons.com/2.5.94/css/materialdesignicons.min.css">
</head>
<body>
	<script type="text/javascript" src="https://www.termsfeed.com/public/cookie-consent/4.1.0/cookie-consent.js" charset="UTF-8"></script>
<script type="text/javascript" charset="UTF-8">
document.addEventListener('DOMContentLoaded', function () {
cookieconsent.run({"notice_banner_type":"simple","consent_type":"express","palette":"light","language":"en","page_load_consent_levels":["strictly-necessary"],"notice_banner_reject_button_hide":false,"preferences_center_close_button_hide":false,"page_refresh_confirmation_buttons":false});
});
</script>

<!-- Google Analytics -->
<script type="text/plain" data-cookie-consent="tracking" async src="https://www.googletagmanager.com/gtag/js?id=G-THG9Q3WPEJ"></script>
<script type="text/plain" data-cookie-consent="tracking">
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-THG9Q3WPEJ');
</script>
<!-- end of Google Analytics-->
	
	<noscript>Free cookie consent management tool by <a href="https://www.termsfeed.com/">TermsFeed</a></noscript>
	<!-- End Cookie Consent by TermsFeed https://www.TermsFeed.com -->
<header>
	<div id="header-img"><a href="https://2024.webgestalt.org"><img border="0" src="https://2024.webgestalt.org/images/gsa.png"></a></div>
	<div id="header-title">
		<p id="header-title-text">WEB-based GEne SeT AnaLysis Toolkit</p>
		<p id="header-subtitle-text">Translating gene lists into biological insights...</p>
	</div>
</header>
<hr><main>
<div id="app">
<div class="loading-preinit" v-cloak><div class="loading-icon"></div></div>
<div class="wrapper" v-cloak>
<div class="level">
	<div class="level-left">
		<div class="level-item">
			<h4 class="title is-4 has-text-primary">Summary</h4>
		</div>
	</div>
	<div class="level-right">
		<div class="level-item">
			<a class="button is-text" href="Project_wg_result1720045977.zip" target="_blank">Result Download</a>
		</div>
	</div>
</div>

<section id="summary" class="summary">
<div class="card" v-bind:class="{active: summaryShown}" >
	<div class="card-header" v-on:click="summaryShown=!summaryShown">
		<p class="card-header-title">Job summary</p>
		<a href="#" class="card-header-icon">
			<b-icon :icon="summaryShown ? 'chevron-up' : 'chevron-down'"></b-icon>
		</a>
	</div>
	<div class="card-content content" v-show="summaryShown">
		<ul id="summary-list"> 
		<li><b>Enrichment method:</b> ORA</li>
		<li><b>Organism: </b>hsapiens</li>
		<li>
				<b>Enrichment Categories: </b>pathway_KEGG
				<a class="has-text-info" href="https://2024.webgestalt.org/api/geneset?organism=hsapiens&database=pathway_KEGG&fileType=gmt">
					<b-icon icon="download" size="is-small"></b-icon>
				</a>
		</li>

			<li><b>Interesting list: </b>textAreaUpload_1720045977.txt. <b>ID type: </b>genesymbol</li>
			<li>The interesting list contains <b>22</b> user IDs in which <b>20</b> user IDs are unambiguously mapped to <b>20</b> unique entrezgene IDs and <b>2</b> user IDs can not be mapped to any entrezgene ID.</li>
			<li>The GO Slim summary are based upon the <b>20</b> unique entrezgene IDs.</li>
		<li>Among <b>20</b> unique entrezgene IDs, <b>11</b> IDs are annotated to the selected functional categories and also in the reference list, which are used for the enrichment analysis.</li>
		<li><b>Reference list: </b> all mapped entrezgene IDs from the selected platform genome
			<a class="has-text-info" href="https://2024.webgestalt.org/api/reference?organism=hsapiens&referenceSet=genome">
				<b-icon icon="download" size="is-small"></b-icon>
			</a>
		</li>
	<li>The reference list can be mapped to <b>193394</b> entrezgene IDs and <b> 8644</b> IDs are annotated to the selected functional categories that are used as the reference for the enrichment analysis.</li>
</ul> 

<p><b>Parameters for the enrichment analysis:</b></p>
<ul><li><b>Minimum number of IDs in the category: </b>5</li>
<li><b>Maximum number of IDs in the category: </b>2000</li>
<li><b>FDR Method: </b>BH</li>
<li><b>Significance Level:</b> Top 10</li>
</ul>
	<p>Based on the above parameters, <b>10</b> categories are identified as enriched categories and all are shown in this report.</p>

	</div>
</div>
</section>
<section id="go-summary" class="summary">
<div class="card" v-bind:class="{active: goShown}" >
	<div class="card-header" v-on:click="goShown=!goShown">
		<p class="card-header-title">GO Slim summary for the user uploaded IDs</p>
		<a href="#" class="card-header-icon">
			<b-icon :icon="goShown ? 'chevron-up' : 'chevron-down'"></b-icon>
		</a>
	</div>
	<div v-show="goShown">
		<div class="card-content">
			<p>Each Biological Process, Cellular Component and Molecular Function category is represented by a red, blue and green bar, repectively.</p>
			<p>The height of the bar represents the number of IDs in the user list and also in the category.</p>
		</div>
		<div class="card-image">
			<img src="goslim_summary_wg_result1720045977.png" width="100%" height="100%" alt="Bar charts of enriched GO Slim terms"/>
		</div>
	</div>
</div>
</section> <h4 class="title is-4 has-text-primary">Enrichment Results</h4>
<section id="enrich-result">
	<p class="select-redundancy">
		<b-tooltip label="Enriched analyte sets are post-processed by two methods to reduce redundancy." type="is-dark" animated multilined dashed>
			Redundancy reduction:
		</b-tooltip>
		<b-radio v-model="redundancy" native-value="all">None</b-radio>
		<b-radio v-model="redundancy" native-value="wsc">
			<b-tooltip label="Find top analyte sets while maximizing analyte coverage." type="is-dark" animated multilined dashed>
				Weighted set cover
			</b-tooltip>
		</b-radio>
	</p>
	<b-tabs v-model="activeTab" type="is-toggle">
		<b-tab-item label="Table">
			<wg-result-table
				:data="enrichmentData"
				:method="enrichMethod"
			></wg-result-table>
		</b-tab-item>
		<b-tab-item label="Bar chart">
		<div class="vertical-center-container">
			<wg-bar-plot
				:dimensions="bar.dimensions"
				:label-width="bar.labelWidth"
				:data="enrichmentData"
				:value-key="bar.valueKey"
				:x-label="bar.xLabel">
			</wg-bar-plot>
		</div>
		</b-tab-item>
		<b-tab-item label="Volcano plot">
		<div class="vertical-center-container">
			<wg-volcano-plot
				:data="volcanoData"
				:x="volcano.x"
				:y="volcano.y"
				:dimensions="volcano.dimensions"
				:x-label="volcano.xLabel"
				:label-types="volcano.labelTypes"
				:x-transform="volcano.xTransform"
				:radius="volcano.radius"
				:color="volcano.color"
				:tooltip-content="volcano.tooltipContent"
				:show-all-labels="false"
				:auto-place-labels="true"
			></wg-volcano-plot>
		</div>
		</b-tab-item>
	</b-tabs>
	<wg-gene-table
		:data="geneTables"
		:info="enrichment"
		:standard-id="standardId"
		:method="enrichMethod"
		:mapped-input-size="mappedInputSize"
		:organism="organism"
		:host-name="hostName"
		:sets-with-net="false"
	></wg-gene-table>
</section> 

</div>
</div>
</main>
	<script type="text/javascript" src="https://2024.webgestalt.org/assets/1.0/build/wg.min.js"></script>
<script type="text/javascript">
var enrichment = [{"geneSet":"hsa01230","description":"Biosynthesis of amino acids","link":"http://www.kegg.jp/kegg-bin/show_pathway?hsa01230+29968+440","size":75,"overlap":2,"expect":0.095441925034706152,"enrichmentRatio":20.955151515151517,"pValue":0.0038838120719689243,"FDR":0.89329378207348242,"overlapId":"29968;440"},{"geneSet":"hsa05150","description":"Staphylococcus aureus infection","link":"http://www.kegg.jp/kegg-bin/show_pathway?hsa05150+3426+3860","size":96,"overlap":2,"expect":0.12216566404442387,"enrichmentRatio":16.371212121212121,"pValue":0.0062896519981327208,"FDR":0.89329378207348242,"overlapId":"3426;3860"},{"geneSet":"hsa00750","description":"Vitamin B6 metabolism","link":"http://www.kegg.jp/kegg-bin/show_pathway?hsa00750+29968","size":6,"overlap":1,"expect":0.0076353540027764922,"enrichmentRatio":130.96969696969697,"pValue":0.0076132992790353621,"FDR":0.89329378207348242,"overlapId":"29968"},{"geneSet":"hsa01240","description":"Biosynthesis of cofactors","link":"http://www.kegg.jp/kegg-bin/show_pathway?hsa01240+29968+9249","size":153,"overlap":2,"expect":0.19470152707080055,"enrichmentRatio":10.272133095662507,"pValue":0.015416237534778346,"FDR":1,"overlapId":"29968;9249"},{"geneSet":"hsa00500","description":"Starch and sucrose metabolism","link":"http://www.kegg.jp/kegg-bin/show_pathway?hsa00500+5837","size":36,"overlap":1,"expect":0.045812124016658955,"enrichmentRatio":21.828282828282827,"pValue":0.044895403430533616,"FDR":1,"overlapId":"5837"},{"geneSet":"hsa00250","description":"Alanine, aspartate and glutamate metabolism","link":"http://www.kegg.jp/kegg-bin/show_pathway?hsa00250+440","size":37,"overlap":1,"expect":0.047084683017121702,"enrichmentRatio":21.238329238329239,"pValue":0.046115913485749838,"FDR":1,"overlapId":"440"},{"geneSet":"hsa04960","description":"Aldosterone-regulated sodium reabsorption","link":"http://www.kegg.jp/kegg-bin/show_pathway?hsa04960+6337","size":37,"overlap":1,"expect":0.047084683017121702,"enrichmentRatio":21.238329238329239,"pValue":0.046115913485749838,"FDR":1,"overlapId":"6337"},{"geneSet":"hsa00260","description":"Glycine, serine and threonine metabolism","link":"http://www.kegg.jp/kegg-bin/show_pathway?hsa00260+29968","size":40,"overlap":1,"expect":0.050902360018509951,"enrichmentRatio":19.645454545454545,"pValue":0.04976894132250697,"FDR":1,"overlapId":"29968"},{"geneSet":"hsa04216","description":"Ferroptosis","link":"http://www.kegg.jp/kegg-bin/show_pathway?hsa04216+1356","size":41,"overlap":1,"expect":0.052174919018972699,"enrichmentRatio":19.166297117516628,"pValue":0.050983788097360247,"FDR":1,"overlapId":"1356"},{"geneSet":"hsa05219","description":"Bladder cancer","link":"http://www.kegg.jp/kegg-bin/show_pathway?hsa05219+3576","size":41,"overlap":1,"expect":0.052174919018972699,"enrichmentRatio":19.166297117516628,"pValue":0.050983788097360247,"FDR":1,"overlapId":"3576"}];
var background = [{"geneSet":"hsa04060","enrichmentRatio":5.2917049280685644,"FDR":1,"overlap":2},{"geneSet":"hsa00860","enrichmentRatio":18.274841437632137,"FDR":1,"overlap":1},{"geneSet":"hsa05144","enrichmentRatio":15.716363636363635,"FDR":1,"overlap":1},{"geneSet":"hsa00270","enrichmentRatio":15.111888111888112,"FDR":1,"overlap":1},{"geneSet":"hsa05134","enrichmentRatio":14.032467532467534,"FDR":1,"overlap":1},{"geneSet":"hsa00830","enrichmentRatio":11.55614973262032,"FDR":1,"overlap":1},{"geneSet":"hsa05120","enrichmentRatio":11.225974025974025,"FDR":1,"overlap":1},{"geneSet":"hsa04622","enrichmentRatio":11.067861715749039,"FDR":1,"overlap":1},{"geneSet":"hsa05133","enrichmentRatio":10.339712918660288,"FDR":1,"overlap":1},{"geneSet":"hsa04610","enrichmentRatio":9.1374207188160685,"FDR":1,"overlap":1},{"geneSet":"hsa04742","enrichmentRatio":9.1374207188160685,"FDR":1,"overlap":1},{"geneSet":"hsa05323","enrichmentRatio":8.4496578690127073,"FDR":1,"overlap":1},{"geneSet":"hsa04657","enrichmentRatio":8.359767891682786,"FDR":1,"overlap":1},{"geneSet":"hsa01100","enrichmentRatio":2.033164765376926,"FDR":1,"overlap":4},{"geneSet":"hsa04061","enrichmentRatio":7.8581818181818175,"FDR":1,"overlap":1},{"geneSet":"hsa04933","enrichmentRatio":7.8581818181818175,"FDR":1,"overlap":1},{"geneSet":"hsa05142","enrichmentRatio":7.7040998217468806,"FDR":1,"overlap":1},{"geneSet":"hsa05146","enrichmentRatio":7.7040998217468806,"FDR":1,"overlap":1},{"geneSet":"hsa04064","enrichmentRatio":7.5559440559440558,"FDR":1,"overlap":1},{"geneSet":"hsa04922","enrichmentRatio":7.3440951571792699,"FDR":1,"overlap":1},{"geneSet":"hsa04620","enrichmentRatio":7.2760942760942759,"FDR":1,"overlap":1},{"geneSet":"hsa04931","enrichmentRatio":7.2760942760942759,"FDR":1,"overlap":1},{"geneSet":"hsa01200","enrichmentRatio":6.8332015810276676,"FDR":1,"overlap":1},{"geneSet":"hsa04910","enrichmentRatio":5.7358991373589907,"FDR":1,"overlap":1},{"geneSet":"hsa04915","enrichmentRatio":5.7358991373589907,"FDR":1,"overlap":1},{"geneSet":"hsa05135","enrichmentRatio":5.7358991373589907,"FDR":1,"overlap":1},{"geneSet":"hsa04936","enrichmentRatio":5.5339308578745197,"FDR":1,"overlap":1},{"geneSet":"hsa04072","enrichmentRatio":5.3095823095823098,"FDR":1,"overlap":1},{"geneSet":"hsa04932","enrichmentRatio":5.0697947214076251,"FDR":1,"overlap":1},{"geneSet":"hsa04218","enrichmentRatio":5.0372960372960369,"FDR":1,"overlap":1},{"geneSet":"hsa04217","enrichmentRatio":4.942252715837621,"FDR":1,"overlap":1},{"geneSet":"hsa05161","enrichmentRatio":4.850729517396184,"FDR":1,"overlap":1},{"geneSet":"hsa05164","enrichmentRatio":4.5954279638490165,"FDR":1,"overlap":1},{"geneSet":"hsa04310","enrichmentRatio":4.5161964472309304,"FDR":1,"overlap":1},{"geneSet":"hsa04621","enrichmentRatio":4.2248289345063537,"FDR":1,"overlap":1},{"geneSet":"hsa04062","enrichmentRatio":4.0928030303030303,"FDR":1,"overlap":1},{"geneSet":"hsa05202","enrichmentRatio":4.0715967969853981,"FDR":1,"overlap":1},{"geneSet":"hsa05167","enrichmentRatio":4.0506091846298036,"FDR":1,"overlap":1},{"geneSet":"hsa05130","enrichmentRatio":3.9687786960514231,"FDR":1,"overlap":1},{"geneSet":"hsa05417","enrichmentRatio":3.6549682875264273,"FDR":1,"overlap":1},{"geneSet":"hsa05163","enrichmentRatio":3.4925252525252528,"FDR":1,"overlap":1},{"geneSet":"hsa05171","enrichmentRatio":3.3871473354231973,"FDR":1,"overlap":1},{"geneSet":"hsa05131","enrichmentRatio":3.1814501288185499,"FDR":1,"overlap":1},{"geneSet":"hsa05132","enrichmentRatio":3.1558963125228185,"FDR":1,"overlap":1},{"geneSet":"hsa05200","enrichmentRatio":1.479883581578497,"FDR":1,"overlap":1}];
var geneTables = {"hsa01230":[{"userId":"PSAT1","geneSymbol":"PSAT1","geneName":"phosphoserine aminotransferase 1","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=29968","entrezgene":"29968"},{"userId":"ASNS","geneSymbol":"ASNS","geneName":"asparagine synthetase (glutamine-hydrolyzing)","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=440","entrezgene":"440"}],"hsa05150":[{"userId":"KRT13","geneSymbol":"KRT13","geneName":"keratin 13","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=3860","entrezgene":"3860"},{"userId":"CFI","geneSymbol":"CFI","geneName":"complement factor I","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=3426","entrezgene":"3426"}],"hsa00750":[{"userId":"PSAT1","geneSymbol":"PSAT1","geneName":"phosphoserine aminotransferase 1","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=29968","entrezgene":"29968"}],"hsa01240":[{"userId":"DHRS3","geneSymbol":"DHRS3","geneName":"dehydrogenase/reductase 3","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=9249","entrezgene":"9249"},{"userId":"PSAT1","geneSymbol":"PSAT1","geneName":"phosphoserine aminotransferase 1","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=29968","entrezgene":"29968"}],"hsa00500":[{"userId":"PYGM","geneSymbol":"PYGM","geneName":"glycogen phosphorylase, muscle associated","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=5837","entrezgene":"5837"}],"hsa00250":[{"userId":"ASNS","geneSymbol":"ASNS","geneName":"asparagine synthetase (glutamine-hydrolyzing)","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=440","entrezgene":"440"}],"hsa04960":[{"userId":"SCNN1A","geneSymbol":"SCNN1A","geneName":"sodium channel epithelial 1 subunit alpha","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=6337","entrezgene":"6337"}],"hsa00260":[{"userId":"PSAT1","geneSymbol":"PSAT1","geneName":"phosphoserine aminotransferase 1","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=29968","entrezgene":"29968"}],"hsa04216":[{"userId":"CP","geneSymbol":"CP","geneName":"ceruloplasmin","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=1356","entrezgene":"1356"}],"hsa05219":[{"userId":"CXCL8","geneSymbol":"CXCL8","geneName":"C-X-C motif chemokine ligand 8","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=3576","entrezgene":"3576"}]};
var clusters = {"wsc":{"representatives":["hsa01230","hsa05150","hsa01240","hsa00500","hsa04960","hsa04216","hsa05219"],"coverage":[1]}};

wgLib({
	enrichDatabase: "pathway_KEGG",
	organism: "hsapiens",
	hostName: "https://2024.webgestalt.org",
	enrichment: enrichment,
	background: background,
	geneTables: geneTables,
	clusters: clusters,
	enrichMethod: "ora",
	mappedInputSize: 11,
	standardId: "entrezgene",
	volcano: {
		labelTypes: ["geneSet", "description"],
		dimensions: { width: 960, height: 500 },
		autoPlaceLabels: true,
		y: "FDR",

		x: "enrichmentRatio",
		xLabel: "log2 of enrichment ratio",
		radius: "overlap",
		color: "overlap",
			tooltipContent: ["geneSet", "description", "enrichmentRatio", "FDR", "pValue", "overlap"]
	},
	bar: {
		dimensions: { width: 960, height: 300 },
		labelWidth: 200,
		tooltipContent: ["geneSet", "description", "enrichmentRatio", "FDR", "pValue", "overlap"],
		valueKey: "enrichmentRatio",
		xLabel: "Enrichment ratio"
	}
})


</script>
<footer class="footer has-text-centered">
<p>WebGestalt is currently developed and maintained by Yuxing Liao, Suhas Vasaikar, Zhiao Shi and Bing Zhang at the <a target="_new" href=http://www.zhang-lab.org> Zhang Lab</a>. Other people who have made significant contribution to the project include Jing Wang, Dexter Duncan, Stefan Kirov and Jay Snoddy.</p>
<p><b>Funding credits:</b> NIH/NCI (U24 CA210954); Leidos (15X038); CPRIT (RR160027); NIH/NIAAA (U01 AA016662, U01 AA013512); NIH/NIDA (P01 DA015027); NIH/NIMH (P50 MH078028, P50 MH096972); NIH/NCI (U24 CA159988); NIH/NIGMS (R01 GM088822).
</p>
</footer></body>
</html>