<!DOCTYPE html>
<html>
<head>
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta charset="UTF-8">
<title>WebGestalt (WEB-based GEne SeT AnaLysis Toolkit)</title>
<link rel="icon" type="image/png" href="https://s3-us-west-2.amazonaws.com/webgestalt/favicon.png">

<script src="https://polyfill.io/v3/polyfill.min.js?flags=gated&features=es2015%2Ces2016%2Cfetch"></script>
<script src="https://2024.webgestalt.org/assets/1.0/build/vendor.min.js"></script>
	<script>window.cytoscape = null;</script>
	<link rel="stylesheet" href="https://2024.webgestalt.org/assets/1.0/build/vendor.css">
	<link rel="stylesheet" href="https://2024.webgestalt.org/assets/1.0/build/wg.css">
<link rel="stylesheet" href="https://cdn.materialdesignicons.com/2.5.94/css/materialdesignicons.min.css">
</head>
<body>
	<script type="text/javascript" src="https://www.termsfeed.com/public/cookie-consent/4.1.0/cookie-consent.js" charset="UTF-8"></script>
<script type="text/javascript" charset="UTF-8">
document.addEventListener('DOMContentLoaded', function () {
cookieconsent.run({"notice_banner_type":"simple","consent_type":"express","palette":"light","language":"en","page_load_consent_levels":["strictly-necessary"],"notice_banner_reject_button_hide":false,"preferences_center_close_button_hide":false,"page_refresh_confirmation_buttons":false});
});
</script>

<!-- Google Analytics -->
<script type="text/plain" data-cookie-consent="tracking" async src="https://www.googletagmanager.com/gtag/js?id=G-THG9Q3WPEJ"></script>
<script type="text/plain" data-cookie-consent="tracking">
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-THG9Q3WPEJ');
</script>
<!-- end of Google Analytics-->
	
	<noscript>Free cookie consent management tool by <a href="https://www.termsfeed.com/">TermsFeed</a></noscript>
	<!-- End Cookie Consent by TermsFeed https://www.TermsFeed.com -->
<header>
	<div id="header-img"><a href="https://2024.webgestalt.org"><img border="0" src="https://2024.webgestalt.org/images/gsa.png"></a></div>
	<div id="header-title">
		<p id="header-title-text">WEB-based GEne SeT AnaLysis Toolkit</p>
		<p id="header-subtitle-text">Translating gene lists into biological insights...</p>
	</div>
</header>
<hr><main>
<div id="app">
<div class="loading-preinit" v-cloak><div class="loading-icon"></div></div>
<div class="wrapper" v-cloak>
<div class="level">
	<div class="level-left">
		<div class="level-item">
			<h4 class="title is-4 has-text-primary">Summary</h4>
		</div>
	</div>
	<div class="level-right">
		<div class="level-item">
			<a class="button is-text" href="Project_wg_result1720045529.zip" target="_blank">Result Download</a>
		</div>
	</div>
</div>

<section id="summary" class="summary">
<div class="card" v-bind:class="{active: summaryShown}" >
	<div class="card-header" v-on:click="summaryShown=!summaryShown">
		<p class="card-header-title">Job summary</p>
		<a href="#" class="card-header-icon">
			<b-icon :icon="summaryShown ? 'chevron-up' : 'chevron-down'"></b-icon>
		</a>
	</div>
	<div class="card-content content" v-show="summaryShown">
		<ul id="summary-list"> 
		<li><b>Enrichment method:</b> ORA</li>
		<li><b>Organism: </b>hsapiens</li>
		<li>
				<b>Enrichment Categories: </b>pathway_Panther
				<a class="has-text-info" href="https://2024.webgestalt.org/api/geneset?organism=hsapiens&database=pathway_Panther&fileType=gmt">
					<b-icon icon="download" size="is-small"></b-icon>
				</a>
		</li>

			<li><b>Interesting list: </b>textAreaUpload_1720045529.txt. <b>ID type: </b>genesymbol</li>
			<li>The interesting list contains <b>584</b> user IDs in which <b>510</b> user IDs are unambiguously mapped to <b>510</b> unique entrezgene IDs and <b>74</b> user IDs can not be mapped to any entrezgene ID.</li>
			<li>The GO Slim summary are based upon the <b>510</b> unique entrezgene IDs.</li>
		<li>Among <b>510</b> unique entrezgene IDs, <b>74</b> IDs are annotated to the selected functional categories and also in the reference list, which are used for the enrichment analysis.</li>
		<li><b>Reference list: </b> all mapped entrezgene IDs from the selected platform genome
			<a class="has-text-info" href="https://2024.webgestalt.org/api/reference?organism=hsapiens&referenceSet=genome">
				<b-icon icon="download" size="is-small"></b-icon>
			</a>
		</li>
	<li>The reference list can be mapped to <b>193394</b> entrezgene IDs and <b> 2256</b> IDs are annotated to the selected functional categories that are used as the reference for the enrichment analysis.</li>
</ul> 

<p><b>Parameters for the enrichment analysis:</b></p>
<ul><li><b>Minimum number of IDs in the category: </b>5</li>
<li><b>Maximum number of IDs in the category: </b>2000</li>
<li><b>FDR Method: </b>BH</li>
<li><b>Significance Level:</b> Top 10</li>
</ul>
	<p>Based on the above parameters, <b>10</b> categories are identified as enriched categories and all are shown in this report.</p>

	</div>
</div>
</section>
<section id="go-summary" class="summary">
<div class="card" v-bind:class="{active: goShown}" >
	<div class="card-header" v-on:click="goShown=!goShown">
		<p class="card-header-title">GO Slim summary for the user uploaded IDs</p>
		<a href="#" class="card-header-icon">
			<b-icon :icon="goShown ? 'chevron-up' : 'chevron-down'"></b-icon>
		</a>
	</div>
	<div v-show="goShown">
		<div class="card-content">
			<p>Each Biological Process, Cellular Component and Molecular Function category is represented by a red, blue and green bar, repectively.</p>
			<p>The height of the bar represents the number of IDs in the user list and also in the category.</p>
		</div>
		<div class="card-image">
			<img src="goslim_summary_wg_result1720045529.png" width="100%" height="100%" alt="Bar charts of enriched GO Slim terms"/>
		</div>
	</div>
</div>
</section> <h4 class="title is-4 has-text-primary">Enrichment Results</h4>
<section id="enrich-result">
	<p class="select-redundancy">
		<b-tooltip label="Enriched analyte sets are post-processed by two methods to reduce redundancy." type="is-dark" animated multilined dashed>
			Redundancy reduction:
		</b-tooltip>
		<b-radio v-model="redundancy" native-value="all">None</b-radio>
		<b-radio v-model="redundancy" native-value="wsc">
			<b-tooltip label="Find top analyte sets while maximizing analyte coverage." type="is-dark" animated multilined dashed>
				Weighted set cover
			</b-tooltip>
		</b-radio>
	</p>
	<b-tabs v-model="activeTab" type="is-toggle">
		<b-tab-item label="Table">
			<wg-result-table
				:data="enrichmentData"
				:method="enrichMethod"
			></wg-result-table>
		</b-tab-item>
		<b-tab-item label="Bar chart">
		<div class="vertical-center-container">
			<wg-bar-plot
				:dimensions="bar.dimensions"
				:label-width="bar.labelWidth"
				:data="enrichmentData"
				:value-key="bar.valueKey"
				:x-label="bar.xLabel">
			</wg-bar-plot>
		</div>
		</b-tab-item>
		<b-tab-item label="Volcano plot">
		<div class="vertical-center-container">
			<wg-volcano-plot
				:data="volcanoData"
				:x="volcano.x"
				:y="volcano.y"
				:dimensions="volcano.dimensions"
				:x-label="volcano.xLabel"
				:label-types="volcano.labelTypes"
				:x-transform="volcano.xTransform"
				:radius="volcano.radius"
				:color="volcano.color"
				:tooltip-content="volcano.tooltipContent"
				:show-all-labels="false"
				:auto-place-labels="true"
			></wg-volcano-plot>
		</div>
		</b-tab-item>
	</b-tabs>
	<wg-gene-table
		:data="geneTables"
		:info="enrichment"
		:standard-id="standardId"
		:method="enrichMethod"
		:mapped-input-size="mappedInputSize"
		:organism="organism"
		:host-name="hostName"
		:sets-with-net="false"
	></wg-gene-table>
</section> 

</div>
</div>
</main>
	<script type="text/javascript" src="https://2024.webgestalt.org/assets/1.0/build/wg.min.js"></script>
<script type="text/javascript">
var enrichment = [{"geneSet":"P02776","description":"Serine glycine biosynthesis","link":"http://www.pantherdb.org/pathway/pathwayDiagram.jsp?catAccession=P02776","size":5,"overlap":2,"expect":0.16400709219858156,"enrichmentRatio":12.194594594594594,"pValue":0.0099561548776512035,"FDR":1,"overlapId":"26227;29968"},{"geneSet":"P00017","description":"DNA replication","link":"http://www.pantherdb.org/pathway/pathwayDiagram.jsp?catAccession=P00017","size":19,"overlap":3,"expect":0.62322695035460995,"enrichmentRatio":4.8136557610241821,"pValue":0.022502100202364109,"FDR":1,"overlapId":"7153;5557;1763"},{"geneSet":"P00059","description":"p53 pathway","link":"http://www.pantherdb.org/pathway/pathwayDiagram.jsp?catAccession=P00059","size":79,"overlap":6,"expect":2.5913120567375887,"enrichmentRatio":2.3154293534040371,"pValue":0.042442039259278967,"FDR":1,"overlapId":"983;891;995;6241;1026;51512"},{"geneSet":"P00044","description":"Nicotinic acetylcholine receptor signaling pathway","link":"http://www.pantherdb.org/pathway/pathwayDiagram.jsp?catAccession=P00044","size":92,"overlap":5,"expect":3.0177304964539009,"enrichmentRatio":1.6568742655699176,"pValue":0.18126754844561757,"FDR":1,"overlapId":"4621;1138;59;4647;53904"},{"geneSet":"P02727","description":"Androgen/estrogene/progesterone biosynthesis","link":"http://www.pantherdb.org/pathway/pathwayDiagram.jsp?catAccession=P02727","size":9,"overlap":1,"expect":0.29521276595744683,"enrichmentRatio":3.387387387387387,"pValue":0.25970430198973093,"FDR":1,"overlapId":"8630"},{"geneSet":"P00020","description":"FAS signaling pathway","link":"http://www.pantherdb.org/pathway/pathwayDiagram.jsp?catAccession=P00020","size":31,"overlap":2,"expect":1.0168439716312057,"enrichmentRatio":1.966870095902354,"pValue":0.27046112216302753,"FDR":1,"overlapId":"4001;822"},{"geneSet":"P02771","description":"Pyrimidine Metabolism","link":"http://www.pantherdb.org/pathway/pathwayDiagram.jsp?catAccession=P02771","size":10,"overlap":1,"expect":0.32801418439716312,"enrichmentRatio":3.0486486486486486,"pValue":0.28408431162667097,"FDR":1,"overlapId":"51733"},{"geneSet":"P00036","description":"Interleukin signaling pathway","link":"http://www.pantherdb.org/pathway/pathwayDiagram.jsp?catAccession=P00036","size":86,"overlap":4,"expect":2.8209219858156027,"enrichmentRatio":1.4179761156505344,"pValue":0.31105886553406059,"FDR":1,"overlapId":"3576;1026;6776;3590"},{"geneSet":"P02739","description":"De novo pyrimidine deoxyribonucleotide biosynthesis","link":"http://www.pantherdb.org/pathway/pathwayDiagram.jsp?catAccession=P02739","size":12,"overlap":1,"expect":0.39361702127659576,"enrichmentRatio":2.5405405405405403,"pValue":0.33049253055764599,"FDR":1,"overlapId":"6241"},{"geneSet":"P00045","description":"Notch signaling pathway","link":"http://www.pantherdb.org/pathway/pathwayDiagram.jsp?catAccession=P00045","size":40,"overlap":2,"expect":1.3120567375886525,"enrichmentRatio":1.5243243243243243,"pValue":0.3801653728190556,"FDR":1,"overlapId":"84708;23462"}];
var background = [{"geneSet":"P00012","enrichmentRatio":1.1955484896661368,"FDR":1,"overlap":6},{"geneSet":"P00038","enrichmentRatio":2.0324324324324325,"FDR":1,"overlap":1},{"geneSet":"P00016","enrichmentRatio":1.3065637065637066,"FDR":1,"overlap":3},{"geneSet":"P00050","enrichmentRatio":1.9054054054054055,"FDR":1,"overlap":1},{"geneSet":"P00060","enrichmentRatio":1.3857493857493859,"FDR":1,"overlap":2},{"geneSet":"P04398","enrichmentRatio":1.3254994124559341,"FDR":1,"overlap":2},{"geneSet":"P04397","enrichmentRatio":1.4517374517374517,"FDR":1,"overlap":1},{"geneSet":"P00002","enrichmentRatio":1.3254994124559341,"FDR":1,"overlap":1},{"geneSet":"P00049","enrichmentRatio":1.0512581547064306,"FDR":1,"overlap":3},{"geneSet":"P00041","enrichmentRatio":1.2702702702702702,"FDR":1,"overlap":1},{"geneSet":"P00030","enrichmentRatio":1.1725571725571726,"FDR":1,"overlap":1},{"geneSet":"P00005","enrichmentRatio":0.97713097713097707,"FDR":1,"overlap":5},{"geneSet":"P00001","enrichmentRatio":1.1291291291291292,"FDR":1,"overlap":1},{"geneSet":"P02738","enrichmentRatio":1.1291291291291292,"FDR":1,"overlap":1},{"geneSet":"P06587","enrichmentRatio":1.0888030888030888,"FDR":1,"overlap":1},{"geneSet":"P00057","enrichmentRatio":0.93965198074787104,"FDR":1,"overlap":9},{"geneSet":"P00034","enrichmentRatio":0.9294660514172709,"FDR":1,"overlap":5},{"geneSet":"P00031","enrichmentRatio":0.92383292383292381,"FDR":1,"overlap":6},{"geneSet":"P05918","enrichmentRatio":0.92383292383292392,"FDR":1,"overlap":1},{"geneSet":"P00028","enrichmentRatio":0.89666136724960266,"FDR":1,"overlap":1},{"geneSet":"P05911","enrichmentRatio":0.87104247104247112,"FDR":1,"overlap":1},{"geneSet":"P00006","enrichmentRatio":0.85476130335943423,"FDR":1,"overlap":3},{"geneSet":"P00011","enrichmentRatio":0.80227596017069702,"FDR":1,"overlap":1},{"geneSet":"P00004","enrichmentRatio":0.81660231660231652,"FDR":1,"overlap":3},{"geneSet":"P00037","enrichmentRatio":0.69287469287469294,"FDR":1,"overlap":1},{"geneSet":"P00047","enrichmentRatio":0.73757628596338276,"FDR":1,"overlap":3},{"geneSet":"P00029","enrichmentRatio":0.70898805782526708,"FDR":1,"overlap":3},{"geneSet":"P06959","enrichmentRatio":0.71313418681839735,"FDR":1,"overlap":4},{"geneSet":"P00042","enrichmentRatio":0.55429975429975431,"FDR":1,"overlap":1},{"geneSet":"P00056","enrichmentRatio":0.49977846699158174,"FDR":1,"overlap":1},{"geneSet":"P00026","enrichmentRatio":0.6056917844997316,"FDR":1,"overlap":3},{"geneSet":"P00039","enrichmentRatio":0.46902286902286905,"FDR":1,"overlap":1},{"geneSet":"P00018","enrichmentRatio":0.53958383161922985,"FDR":1,"overlap":2},{"geneSet":"P00027","enrichmentRatio":0.52113652113652109,"FDR":1,"overlap":2},{"geneSet":"P04393","enrichmentRatio":0.43552123552123556,"FDR":1,"overlap":1},{"geneSet":"P00053","enrichmentRatio":0.40648648648648644,"FDR":1,"overlap":1},{"geneSet":"P00019","enrichmentRatio":0.39085239085239082,"FDR":1,"overlap":1},{"geneSet":"P00021","enrichmentRatio":0.29598530569404358,"FDR":1,"overlap":1}];
var geneTables = {"P02776":[{"userId":"PHGDH","geneSymbol":"PHGDH","geneName":"phosphoglycerate dehydrogenase","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=26227","entrezgene":"26227"},{"userId":"PSAT1","geneSymbol":"PSAT1","geneName":"phosphoserine aminotransferase 1","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=29968","entrezgene":"29968"}],"P00017":[{"userId":"DNA2","geneSymbol":"DNA2","geneName":"DNA replication helicase/nuclease 2","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=1763","entrezgene":"1763"},{"userId":"PRIM1","geneSymbol":"PRIM1","geneName":"DNA primase subunit 1","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=5557","entrezgene":"5557"},{"userId":"TOP2A","geneSymbol":"TOP2A","geneName":"DNA topoisomerase II alpha","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=7153","entrezgene":"7153"}],"P00059":[{"userId":"CDKN1A","geneSymbol":"CDKN1A","geneName":"cyclin dependent kinase inhibitor 1A","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=1026","entrezgene":"1026"},{"userId":"RRM2","geneSymbol":"RRM2","geneName":"ribonucleotide reductase regulatory subunit M2","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=6241","entrezgene":"6241"},{"userId":"CDC25C","geneSymbol":"CDC25C","geneName":"cell division cycle 25C","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=995","entrezgene":"995"},{"userId":"CDK1","geneSymbol":"CDK1","geneName":"cyclin dependent kinase 1","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=983","entrezgene":"983"},{"userId":"CCNB1","geneSymbol":"CCNB1","geneName":"cyclin B1","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=891","entrezgene":"891"},{"userId":"GTSE1","geneSymbol":"GTSE1","geneName":"G2 and S-phase expressed 1","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=51512","entrezgene":"51512"}],"P00044":[{"userId":"MYH3","geneSymbol":"MYH3","geneName":"myosin heavy chain 3","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=4621","entrezgene":"4621"},{"userId":"ACTA2","geneSymbol":"ACTA2","geneName":"actin alpha 2, smooth muscle","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=59","entrezgene":"59"},{"userId":"MYO3A","geneSymbol":"MYO3A","geneName":"myosin IIIA","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=53904","entrezgene":"53904"},{"userId":"MYO7A","geneSymbol":"MYO7A","geneName":"myosin VIIA","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=4647","entrezgene":"4647"},{"userId":"CHRNA5","geneSymbol":"CHRNA5","geneName":"cholinergic receptor nicotinic alpha 5 subunit","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=1138","entrezgene":"1138"}],"P02727":[{"userId":"HSD17B6","geneSymbol":"HSD17B6","geneName":"hydroxysteroid 17-beta dehydrogenase 6","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=8630","entrezgene":"8630"}],"P00020":[{"userId":"CAPG","geneSymbol":"CAPG","geneName":"capping actin protein, gelsolin like","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=822","entrezgene":"822"},{"userId":"LMNB1","geneSymbol":"LMNB1","geneName":"lamin B1","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=4001","entrezgene":"4001"}],"P02771":[{"userId":"UPB1","geneSymbol":"UPB1","geneName":"beta-ureidopropionase 1","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=51733","entrezgene":"51733"}],"P00036":[{"userId":"IL11RA","geneSymbol":"IL11RA","geneName":"interleukin 11 receptor subunit alpha","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=3590","entrezgene":"3590"},{"userId":"CDKN1A","geneSymbol":"CDKN1A","geneName":"cyclin dependent kinase inhibitor 1A","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=1026","entrezgene":"1026"},{"userId":"CXCL8","geneSymbol":"CXCL8","geneName":"C-X-C motif chemokine ligand 8","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=3576","entrezgene":"3576"},{"userId":"STAT5A","geneSymbol":"STAT5A","geneName":"signal transducer and activator of transcription 5A","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=6776","entrezgene":"6776"}],"P02739":[{"userId":"RRM2","geneSymbol":"RRM2","geneName":"ribonucleotide reductase regulatory subunit M2","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=6241","entrezgene":"6241"}],"P00045":[{"userId":"LNX1","geneSymbol":"LNX1","geneName":"ligand of numb-protein X 1","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=84708","entrezgene":"84708"},{"userId":"HEY1","geneSymbol":"HEY1","geneName":"hes related family bHLH transcription factor with YRPW motif 1","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=23462","entrezgene":"23462"}]};
var clusters = {"wsc":{"representatives":["P00059","P00017","P02776","P00044","P00036","P00020","P00045","P02727","P02771"],"coverage":[1]}};

wgLib({
	enrichDatabase: "pathway_Panther",
	organism: "hsapiens",
	hostName: "https://2024.webgestalt.org",
	enrichment: enrichment,
	background: background,
	geneTables: geneTables,
	clusters: clusters,
	enrichMethod: "ora",
	mappedInputSize: 74,
	standardId: "entrezgene",
	volcano: {
		labelTypes: ["geneSet", "description"],
		dimensions: { width: 960, height: 500 },
		autoPlaceLabels: true,
		y: "FDR",

		x: "enrichmentRatio",
		xLabel: "log2 of enrichment ratio",
		radius: "overlap",
		color: "overlap",
			tooltipContent: ["geneSet", "description", "enrichmentRatio", "FDR", "pValue", "overlap"]
	},
	bar: {
		dimensions: { width: 960, height: 300 },
		labelWidth: 200,
		tooltipContent: ["geneSet", "description", "enrichmentRatio", "FDR", "pValue", "overlap"],
		valueKey: "enrichmentRatio",
		xLabel: "Enrichment ratio"
	}
})


</script>
<footer class="footer has-text-centered">
<p>WebGestalt is currently developed and maintained by Yuxing Liao, Suhas Vasaikar, Zhiao Shi and Bing Zhang at the <a target="_new" href=http://www.zhang-lab.org> Zhang Lab</a>. Other people who have made significant contribution to the project include Jing Wang, Dexter Duncan, Stefan Kirov and Jay Snoddy.</p>
<p><b>Funding credits:</b> NIH/NCI (U24 CA210954); Leidos (15X038); CPRIT (RR160027); NIH/NIAAA (U01 AA016662, U01 AA013512); NIH/NIDA (P01 DA015027); NIH/NIMH (P50 MH078028, P50 MH096972); NIH/NCI (U24 CA159988); NIH/NIGMS (R01 GM088822).
</p>
</footer></body>
</html>