<!DOCTYPE html>
<html>
<head>
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta charset="UTF-8">
<title>WebGestalt (WEB-based GEne SeT AnaLysis Toolkit)</title>
<link rel="icon" type="image/png" href="https://s3-us-west-2.amazonaws.com/webgestalt/favicon.png">

<script src="https://polyfill.io/v3/polyfill.min.js?flags=gated&features=es2015%2Ces2016%2Cfetch"></script>
<script src="https://2024.webgestalt.org/assets/1.0/build/vendor.min.js"></script>
	<script>window.cytoscape = null;</script>
	<link rel="stylesheet" href="https://2024.webgestalt.org/assets/1.0/build/vendor.css">
	<link rel="stylesheet" href="https://2024.webgestalt.org/assets/1.0/build/wg.css">
<link rel="stylesheet" href="https://cdn.materialdesignicons.com/2.5.94/css/materialdesignicons.min.css">
</head>
<body>
	<script type="text/javascript" src="https://www.termsfeed.com/public/cookie-consent/4.1.0/cookie-consent.js" charset="UTF-8"></script>
<script type="text/javascript" charset="UTF-8">
document.addEventListener('DOMContentLoaded', function () {
cookieconsent.run({"notice_banner_type":"simple","consent_type":"express","palette":"light","language":"en","page_load_consent_levels":["strictly-necessary"],"notice_banner_reject_button_hide":false,"preferences_center_close_button_hide":false,"page_refresh_confirmation_buttons":false});
});
</script>

<!-- Google Analytics -->
<script type="text/plain" data-cookie-consent="tracking" async src="https://www.googletagmanager.com/gtag/js?id=G-THG9Q3WPEJ"></script>
<script type="text/plain" data-cookie-consent="tracking">
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-THG9Q3WPEJ');
</script>
<!-- end of Google Analytics-->
	
	<noscript>Free cookie consent management tool by <a href="https://www.termsfeed.com/">TermsFeed</a></noscript>
	<!-- End Cookie Consent by TermsFeed https://www.TermsFeed.com -->
<header>
	<div id="header-img"><a href="https://2024.webgestalt.org"><img border="0" src="https://2024.webgestalt.org/images/gsa.png"></a></div>
	<div id="header-title">
		<p id="header-title-text">WEB-based GEne SeT AnaLysis Toolkit</p>
		<p id="header-subtitle-text">Translating gene lists into biological insights...</p>
	</div>
</header>
<hr><main>
<div id="app">
<div class="loading-preinit" v-cloak><div class="loading-icon"></div></div>
<div class="wrapper" v-cloak>
<div class="level">
	<div class="level-left">
		<div class="level-item">
			<h4 class="title is-4 has-text-primary">Summary</h4>
		</div>
	</div>
	<div class="level-right">
		<div class="level-item">
			<a class="button is-text" href="Project_wg_result1720047288.zip" target="_blank">Result Download</a>
		</div>
	</div>
</div>

<section id="summary" class="summary">
<div class="card" v-bind:class="{active: summaryShown}" >
	<div class="card-header" v-on:click="summaryShown=!summaryShown">
		<p class="card-header-title">Job summary</p>
		<a href="#" class="card-header-icon">
			<b-icon :icon="summaryShown ? 'chevron-up' : 'chevron-down'"></b-icon>
		</a>
	</div>
	<div class="card-content content" v-show="summaryShown">
		<ul id="summary-list"> 
		<li><b>Enrichment method:</b> ORA</li>
		<li><b>Organism: </b>hsapiens</li>
		<li>
				<b>Enrichment Categories: </b>pathway_Panther
				<a class="has-text-info" href="https://2024.webgestalt.org/api/geneset?organism=hsapiens&database=pathway_Panther&fileType=gmt">
					<b-icon icon="download" size="is-small"></b-icon>
				</a>
		</li>

			<li><b>Interesting list: </b>textAreaUpload_1720047288.txt. <b>ID type: </b>genesymbol</li>
			<li>The interesting list contains <b>100</b> user IDs in which <b>94</b> user IDs are unambiguously mapped to <b>94</b> unique entrezgene IDs and <b>6</b> user IDs can not be mapped to any entrezgene ID.</li>
			<li>The GO Slim summary are based upon the <b>94</b> unique entrezgene IDs.</li>
		<li>Among <b>94</b> unique entrezgene IDs, <b>20</b> IDs are annotated to the selected functional categories and also in the reference list, which are used for the enrichment analysis.</li>
		<li><b>Reference list: </b> all mapped entrezgene IDs from the selected platform genome
			<a class="has-text-info" href="https://2024.webgestalt.org/api/reference?organism=hsapiens&referenceSet=genome">
				<b-icon icon="download" size="is-small"></b-icon>
			</a>
		</li>
	<li>The reference list can be mapped to <b>193394</b> entrezgene IDs and <b> 2256</b> IDs are annotated to the selected functional categories that are used as the reference for the enrichment analysis.</li>
</ul> 

<p><b>Parameters for the enrichment analysis:</b></p>
<ul><li><b>Minimum number of IDs in the category: </b>5</li>
<li><b>Maximum number of IDs in the category: </b>2000</li>
<li><b>FDR Method: </b>BH</li>
<li><b>Significance Level:</b> Top 10</li>
</ul>
	<p>Based on the above parameters, <b>10</b> categories are identified as enriched categories and all are shown in this report.</p>

	</div>
</div>
</section>
<section id="go-summary" class="summary">
<div class="card" v-bind:class="{active: goShown}" >
	<div class="card-header" v-on:click="goShown=!goShown">
		<p class="card-header-title">GO Slim summary for the user uploaded IDs</p>
		<a href="#" class="card-header-icon">
			<b-icon :icon="goShown ? 'chevron-up' : 'chevron-down'"></b-icon>
		</a>
	</div>
	<div v-show="goShown">
		<div class="card-content">
			<p>Each Biological Process, Cellular Component and Molecular Function category is represented by a red, blue and green bar, repectively.</p>
			<p>The height of the bar represents the number of IDs in the user list and also in the category.</p>
		</div>
		<div class="card-image">
			<img src="goslim_summary_wg_result1720047288.png" width="100%" height="100%" alt="Bar charts of enriched GO Slim terms"/>
		</div>
	</div>
</div>
</section> <h4 class="title is-4 has-text-primary">Enrichment Results</h4>
<section id="enrich-result">
	<p class="select-redundancy">
		<b-tooltip label="Enriched analyte sets are post-processed by two methods to reduce redundancy." type="is-dark" animated multilined dashed>
			Redundancy reduction:
		</b-tooltip>
		<b-radio v-model="redundancy" native-value="all">None</b-radio>
		<b-radio v-model="redundancy" native-value="wsc">
			<b-tooltip label="Find top analyte sets while maximizing analyte coverage." type="is-dark" animated multilined dashed>
				Weighted set cover
			</b-tooltip>
		</b-radio>
	</p>
	<b-tabs v-model="activeTab" type="is-toggle">
		<b-tab-item label="Table">
			<wg-result-table
				:data="enrichmentData"
				:method="enrichMethod"
			></wg-result-table>
		</b-tab-item>
		<b-tab-item label="Bar chart">
		<div class="vertical-center-container">
			<wg-bar-plot
				:dimensions="bar.dimensions"
				:label-width="bar.labelWidth"
				:data="enrichmentData"
				:value-key="bar.valueKey"
				:x-label="bar.xLabel">
			</wg-bar-plot>
		</div>
		</b-tab-item>
		<b-tab-item label="Volcano plot">
		<div class="vertical-center-container">
			<wg-volcano-plot
				:data="volcanoData"
				:x="volcano.x"
				:y="volcano.y"
				:dimensions="volcano.dimensions"
				:x-label="volcano.xLabel"
				:label-types="volcano.labelTypes"
				:x-transform="volcano.xTransform"
				:radius="volcano.radius"
				:color="volcano.color"
				:tooltip-content="volcano.tooltipContent"
				:show-all-labels="false"
				:auto-place-labels="true"
			></wg-volcano-plot>
		</div>
		</b-tab-item>
	</b-tabs>
	<wg-gene-table
		:data="geneTables"
		:info="enrichment"
		:standard-id="standardId"
		:method="enrichMethod"
		:mapped-input-size="mappedInputSize"
		:organism="organism"
		:host-name="hostName"
		:sets-with-net="false"
	></wg-gene-table>
</section> 

</div>
</div>
</main>
	<script type="text/javascript" src="https://2024.webgestalt.org/assets/1.0/build/wg.min.js"></script>
<script type="text/javascript">
var enrichment = [{"geneSet":"P00024","description":"Glycolysis","link":"http://www.pantherdb.org/pathway/pathwayDiagram.jsp?catAccession=P00024","size":17,"overlap":4,"expect":0.15070921985815602,"enrichmentRatio":26.541176470588237,"pValue":9.9465986894250017e-06,"FDR":0.0011239656519050252,"overlapId":"5230;2597;2023;7167"},{"geneSet":"P02776","description":"Serine glycine biosynthesis","link":"http://www.pantherdb.org/pathway/pathwayDiagram.jsp?catAccession=P02776","size":5,"overlap":1,"expect":0.044326241134751775,"enrichmentRatio":22.559999999999999,"pValue":0.043585222697913886,"FDR":1,"overlapId":"26227"},{"geneSet":"P02775","description":"Salvage pyrimidine ribonucleotides","link":"http://www.pantherdb.org/pathway/pathwayDiagram.jsp?catAccession=P02775","size":11,"overlap":1,"expect":0.097517730496453903,"enrichmentRatio":10.254545454545454,"pValue":0.093506397977449446,"FDR":1,"overlapId":"7378"},{"geneSet":"P02738","description":"De novo purine biosynthesis","link":"http://www.pantherdb.org/pathway/pathwayDiagram.jsp?catAccession=P02738","size":27,"overlap":1,"expect":0.23936170212765959,"enrichmentRatio":4.1777777777777771,"pValue":0.21481168435849429,"FDR":1,"overlapId":"205"},{"geneSet":"P00028","description":"Heterotrimeric G-protein signaling pathway-rod outer segment phototransduction","link":"http://www.pantherdb.org/pathway/pathwayDiagram.jsp?catAccession=P00028","size":34,"overlap":1,"expect":0.30141843971631205,"enrichmentRatio":3.3176470588235296,"pValue":0.26288331353503824,"FDR":1,"overlapId":"8787"},{"geneSet":"P05911","description":"Angiotensin II-stimulated signaling through G proteins and beta-arrestin","link":"http://www.pantherdb.org/pathway/pathwayDiagram.jsp?catAccession=P05911","size":35,"overlap":1,"expect":0.31028368794326239,"enrichmentRatio":3.2228571428571433,"pValue":0.26951802718426582,"FDR":1,"overlapId":"183"},{"geneSet":"P00027","description":"Heterotrimeric G-protein signaling pathway-Gq alpha and Go alpha mediated pathway","link":"http://www.pantherdb.org/pathway/pathwayDiagram.jsp?catAccession=P00027","size":117,"overlap":2,"expect":1.0372340425531914,"enrichmentRatio":1.9282051282051285,"pValue":0.27819448357426985,"FDR":1,"overlapId":"8787;8218"},{"geneSet":"P00011","description":"Blood coagulation","link":"http://www.pantherdb.org/pathway/pathwayDiagram.jsp?catAccession=P00011","size":38,"overlap":1,"expect":0.33687943262411346,"enrichmentRatio":2.9684210526315788,"pValue":0.28908345729698964,"FDR":1,"overlapId":"5627"},{"geneSet":"P00045","description":"Notch signaling pathway","link":"http://www.pantherdb.org/pathway/pathwayDiagram.jsp?catAccession=P00045","size":40,"overlap":1,"expect":0.3546099290780142,"enrichmentRatio":2.8199999999999998,"pValue":0.30184937642451404,"FDR":1,"overlapId":"84708"},{"geneSet":"P00054","description":"Toll receptor signaling pathway","link":"http://www.pantherdb.org/pathway/pathwayDiagram.jsp?catAccession=P00054","size":49,"overlap":1,"expect":0.43439716312056736,"enrichmentRatio":2.3020408163265307,"pValue":0.35664908041112858,"FDR":1,"overlapId":"4792"}];
var background = [{"geneSet":"P00026","enrichmentRatio":1.4940397350993377,"FDR":1,"overlap":2},{"geneSet":"P00010","enrichmentRatio":1.9118644067796611,"FDR":1,"overlap":1},{"geneSet":"P00034","enrichmentRatio":1.3756097560975611,"FDR":1,"overlap":2},{"geneSet":"P00003","enrichmentRatio":1.7904761904761906,"FDR":1,"overlap":1},{"geneSet":"P06959","enrichmentRatio":1.3192982456140352,"FDR":1,"overlap":2},{"geneSet":"P00016","enrichmentRatio":1.6114285714285717,"FDR":1,"overlap":1},{"geneSet":"P00053","enrichmentRatio":1.504,"FDR":1,"overlap":1},{"geneSet":"P00031","enrichmentRatio":1.1393939393939394,"FDR":1,"overlap":2},{"geneSet":"P00044","enrichmentRatio":1.2260869565217392,"FDR":1,"overlap":1},{"geneSet":"P00006","enrichmentRatio":1.0542056074766355,"FDR":1,"overlap":1},{"geneSet":"P00004","enrichmentRatio":1.0071428571428571,"FDR":1,"overlap":1},{"geneSet":"P00029","enrichmentRatio":0.87441860465116283,"FDR":1,"overlap":1},{"geneSet":"P00005","enrichmentRatio":0.72307692307692306,"FDR":1,"overlap":1},{"geneSet":"P00057","enrichmentRatio":0.38630136986301367,"FDR":1,"overlap":1}];
var geneTables = {"P00024":[{"userId":"ENO1","geneSymbol":"ENO1","geneName":"enolase 1","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=2023","entrezgene":"2023"},{"userId":"PGK1","geneSymbol":"PGK1","geneName":"phosphoglycerate kinase 1","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=5230","entrezgene":"5230"},{"userId":"TPI1","geneSymbol":"TPI1","geneName":"triosephosphate isomerase 1","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=7167","entrezgene":"7167"},{"userId":"GAPDH","geneSymbol":"GAPDH","geneName":"glyceraldehyde-3-phosphate dehydrogenase","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=2597","entrezgene":"2597"}],"P02776":[{"userId":"PHGDH","geneSymbol":"PHGDH","geneName":"phosphoglycerate dehydrogenase","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=26227","entrezgene":"26227"}],"P02775":[{"userId":"UPP1","geneSymbol":"UPP1","geneName":"uridine phosphorylase 1","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=7378","entrezgene":"7378"}],"P02738":[{"userId":"AK4","geneSymbol":"AK4","geneName":"adenylate kinase 4","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=205","entrezgene":"205"}],"P00028":[{"userId":"RGS9","geneSymbol":"RGS9","geneName":"regulator of G protein signaling 9","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=8787","entrezgene":"8787"}],"P05911":[{"userId":"AGT","geneSymbol":"AGT","geneName":"angiotensinogen","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=183","entrezgene":"183"}],"P00027":[{"userId":"RGS9","geneSymbol":"RGS9","geneName":"regulator of G protein signaling 9","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=8787","entrezgene":"8787"},{"userId":"CLTCL1","geneSymbol":"CLTCL1","geneName":"clathrin heavy chain like 1","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=8218","entrezgene":"8218"}],"P00011":[{"userId":"PROS1","geneSymbol":"PROS1","geneName":"protein S","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=5627","entrezgene":"5627"}],"P00045":[{"userId":"LNX1","geneSymbol":"LNX1","geneName":"ligand of numb-protein X 1","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=84708","entrezgene":"84708"}],"P00054":[{"userId":"NFKBIA","geneSymbol":"NFKBIA","geneName":"NFKB inhibitor alpha","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=4792","entrezgene":"4792"}]};
var clusters = {"wsc":{"representatives":["P00024","P02776","P00027","P02775","P02738","P05911","P00011","P00045","P00054"],"coverage":[1]}};

wgLib({
	enrichDatabase: "pathway_Panther",
	organism: "hsapiens",
	hostName: "https://2024.webgestalt.org",
	enrichment: enrichment,
	background: background,
	geneTables: geneTables,
	clusters: clusters,
	enrichMethod: "ora",
	mappedInputSize: 20,
	standardId: "entrezgene",
	volcano: {
		labelTypes: ["geneSet", "description"],
		dimensions: { width: 960, height: 500 },
		autoPlaceLabels: true,
		y: "FDR",

		x: "enrichmentRatio",
		xLabel: "log2 of enrichment ratio",
		radius: "overlap",
		color: "overlap",
			tooltipContent: ["geneSet", "description", "enrichmentRatio", "FDR", "pValue", "overlap"]
	},
	bar: {
		dimensions: { width: 960, height: 300 },
		labelWidth: 200,
		tooltipContent: ["geneSet", "description", "enrichmentRatio", "FDR", "pValue", "overlap"],
		valueKey: "enrichmentRatio",
		xLabel: "Enrichment ratio"
	}
})


</script>
<footer class="footer has-text-centered">
<p>WebGestalt is currently developed and maintained by Yuxing Liao, Suhas Vasaikar, Zhiao Shi and Bing Zhang at the <a target="_new" href=http://www.zhang-lab.org> Zhang Lab</a>. Other people who have made significant contribution to the project include Jing Wang, Dexter Duncan, Stefan Kirov and Jay Snoddy.</p>
<p><b>Funding credits:</b> NIH/NCI (U24 CA210954); Leidos (15X038); CPRIT (RR160027); NIH/NIAAA (U01 AA016662, U01 AA013512); NIH/NIDA (P01 DA015027); NIH/NIMH (P50 MH078028, P50 MH096972); NIH/NCI (U24 CA159988); NIH/NIGMS (R01 GM088822).
</p>
</footer></body>
</html>