<!DOCTYPE html>
<html>
<head>
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta charset="UTF-8">
<title>WebGestalt (WEB-based GEne SeT AnaLysis Toolkit)</title>
<link rel="icon" type="image/png" href="https://s3-us-west-2.amazonaws.com/webgestalt/favicon.png">
<script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');
ga('create', 'UA-42098540-1', 'auto');
ga('send', 'pageview');
</script>
<script src="https://polyfill.io/v3/polyfill.min.js?flags=gated&features=es2015%2Ces2016%2Cfetch"></script>
<script src="https://www.webgestalt.org/assets/0.4/build/vendor.min.js"></script>
	<script src="https://www.webgestalt.org/assets/0.4/build/cytoscape.min.js"></script>
		<link rel="stylesheet" href="https://www.webgestalt.org/assets/0.4/build/vendor.css">
	<link rel="stylesheet" href="https://www.webgestalt.org/assets/0.4/build/wg.css">
<link rel="stylesheet" href="https://cdn.materialdesignicons.com/2.5.94/css/materialdesignicons.min.css">
</head>
<body>
<header>
	<div id="header-img"><a href="https://www.webgestalt.org"><img border="0" src="https://www.webgestalt.org/images/gsa.png"></a></div>
	<div id="header-title">
		<p id="header-title-text">WEB-based GEne SeT AnaLysis Toolkit</p>
		<p id="header-subtitle-text">Translating gene lists into biological insights...</p>
	</div>
</header>
<hr><main>
<div id="app">
<div class="loading-preinit" v-cloak><div class="loading-icon"></div></div>
<div class="wrapper" v-cloak>
<div class="level">
	<div class="level-left">
		<div class="level-item">
			<h4 class="title is-4 has-text-primary">Summary</h4>
		</div>
	</div>
	<div class="level-right">
		<div class="level-item">
			<a class="button is-text" href="Project_wg_result1698892159.zip" target="_blank">Result Download</a>
		</div>
	</div>
</div>

<section id="summary" class="summary">
<div class="card" v-bind:class="{active: summaryShown}" >
	<div class="card-header" v-on:click="summaryShown=!summaryShown">
		<p class="card-header-title">Job summary</p>
		<a href="#" class="card-header-icon">
			<b-icon :icon="summaryShown ? 'chevron-up' : 'chevron-down'"></b-icon>
		</a>
	</div>
	<div class="card-content content" v-show="summaryShown">
		<ul id="summary-list"> 
		<li><b>Enrichment method:</b> ORA</li>
		<li><b>Organism: </b>rnorvegicus</li>
		<li>
				<b>Enrichment Categories: </b>geneontology_Molecular_Function
				<a class="has-text-info" href="https://www.webgestalt.org/api/geneset?organism=rnorvegicus&database=geneontology_Molecular_Function&fileType=gmt">
					<b-icon icon="download" size="is-small"></b-icon>
				</a>
					</li>
		
			<li><b>Interesting list: </b>textAreaUpload_1698892159.txt. <b>ID type: </b>genesymbol</li>
						<li>The interesting list contains <b>6</b> user IDs in which <b>6</b> user IDs are unambiguously mapped to <b>6</b> unique entrezgene IDs and <b>0</b> user IDs can not be mapped to any entrezgene ID.</li>
			<li>The GO Slim summary are based upon the <b>6</b> unique entrezgene IDs.</li>
		

		<li>Among <b>6</b> unique entrezgene IDs, <b>4</b> IDs are annotated to the selected functional categories and also in the reference list, which are used for the enrichment analysis.</li>
		
		<li><b>Reference list: </b> all mapped entrezgene IDs from the selected platform genome
			<a class="has-text-info" href="https://www.webgestalt.org/api/reference?organism=rnorvegicus&referenceSet=genome">
				<b-icon icon="download" size="is-small"></b-icon>
			</a>
		</li>
	
	<li>The reference list can be mapped to <b>47958</b> entrezgene IDs and <b> 12739</b> IDs are annotated to the selected functional categories that are used as the reference for the enrichment analysis.</li>
</ul> 

<p><b>Parameters for the enrichment analysis:</b></p>
<ul><li><b>Minimum number of IDs in the category: </b>5</li>
<li><b>Maximum number of IDs in the category: </b>2000</li>
<li><b>FDR Method: </b>BH</li>

<li><b>Significance Level:</b> Top 10</li>
</ul>
	
	<p>Based on the above parameters, <b>10</b> categories are identified as enriched categories and all are shown in this report.</p>
	

	</div>
</div>
</section>
<section id="go-summary" class="summary">
<div class="card" v-bind:class="{active: goShown}" >
	<div class="card-header" v-on:click="goShown=!goShown">
		<p class="card-header-title">GO Slim summary for the user uploaded IDs</p>
		<a href="#" class="card-header-icon">
			<b-icon :icon="goShown ? 'chevron-up' : 'chevron-down'"></b-icon>
		</a>
	</div>
	<div v-show="goShown">
		<div class="card-content">
			<p>Each Biological Process, Cellular Component and Molecular Function category is represented by a red, blue and green bar, repectively.</p>
			<p>The height of the bar represents the number of IDs in the user list and also in the category.</p>
		</div>
		<div class="card-image">
			<img src="goslim_summary_wg_result1698892159.png" width="100%" height="100%" alt="Bar charts of enriched GO Slim terms"/>
		</div>
	</div>
</div>
</section> <h4 class="title is-4 has-text-primary">Enrichment Results</h4>
<section id="enrich-result">
	<p class="select-redundancy">
		<b-tooltip label="Enriched gene sets are post-processed by two methods to reduce redundancy." type="is-dark" animated multilined dashed>
			Redundancy reduction:
		</b-tooltip>
		<b-radio v-model="redundancy" native-value="all">All</b-radio>
		<b-radio v-model="redundancy" native-value="ap">
			<b-tooltip label="R package apcluster is used to cluster gene sets." type="is-dark" animated multilined dashed>
				Affinity propagation
			</b-tooltip>
		</b-radio>
		
		<b-radio v-model="redundancy" native-value="wsc">
			<b-tooltip label="Find top gene sets while maximizing gene coverage." type="is-dark" animated multilined dashed>
				Weighted set cover
			</b-tooltip>
		</b-radio>
	</p>
	<b-tabs v-model="activeTab" type="is-toggle">
		<b-tab-item label="Table">
			<wg-result-table
				:data="enrichmentData"
				:method="enrichMethod"
			></wg-result-table>
		</b-tab-item>
		<b-tab-item label="Bar chart">
		<div class="vertical-center-container">
			<wg-bar-plot
				:dimensions="bar.dimensions"
				:label-width="bar.labelWidth"
				:data="enrichmentData"
				:value-key="bar.valueKey"
				:x-label="bar.xLabel">
			</wg-bar-plot>
		</div>
		</b-tab-item>
		<b-tab-item label="Volcano plot">
		<div class="vertical-center-container">
			<wg-volcano-plot
				:data="volcanoData"
				:x="volcano.x"
				:y="volcano.y"
				:dimensions="volcano.dimensions"
				:x-label="volcano.xLabel"
				:label-types="volcano.labelTypes"
				:x-transform="volcano.xTransform"
				:radius="volcano.radius"
				:color="volcano.color"
				:tooltip-content="volcano.tooltipContent"
				:show-all-labels="false"
				:auto-place-labels="true"
			></wg-volcano-plot>
		</div>
		</b-tab-item>
		<b-tab-item label="DAG">
			<b-field>
				<b-select v-model="activeDagDatabase">
					<option v-for="(value, key) in dag"
						:value="key"
						:key="key"
					>{{key}}</option>
				</b-select>
			</b-field>
			<div style="width: 100%; height: 600px">
			<wg-dag
				container-id="wg_dag"
				:data="dagData"
			></wg-dag>
			</div>
		</b-tab-item>
	</b-tabs>
	<wg-gene-table
		:data="geneTables"
		:info="enrichment"
		:standard-id="standardId"
		:method="enrichMethod"
		:mapped-input-size="mappedInputSize"
		:organism="organism"
		:host-name="hostName"
		:sets-with-net="false"
	></wg-gene-table>
</section> 

</div>
</div>
</main>
	<script type="text/javascript" src="https://www.webgestalt.org/assets/0.4/build/wg.min.js"></script>
<script type="text/javascript">
var enrichment = [{"geneSet":"GO:0004016","description":"adenylate cyclase activity","link":"http://amigo.geneontology.org/amigo/term/GO:0004016","size":10,"overlap":1,"expect":0.00313996389041526,"enrichmentRatio":318.475,"pValue":0.00313663748396964,"FDR":1,"overlapId":"84420"},{"geneSet":"GO:0001530","description":"lipopolysaccharide binding","link":"http://amigo.geneontology.org/amigo/term/GO:0001530","size":14,"overlap":1,"expect":0.00439594944658136,"enrichmentRatio":227.482142857143,"pValue":0.0043892241219462,"FDR":1,"overlapId":"362364"},{"geneSet":"GO:0008483","description":"transaminase activity","link":"http://amigo.geneontology.org/amigo/term/GO:0008483","size":20,"overlap":1,"expect":0.00627992778083052,"enrichmentRatio":159.2375,"pValue":0.00626589030428248,"FDR":1,"overlapId":"64313"},{"geneSet":"GO:0009975","description":"cyclase activity","link":"http://amigo.geneontology.org/amigo/term/GO:0009975","size":21,"overlap":1,"expect":0.00659392416987205,"enrichmentRatio":151.654761904762,"pValue":0.00657840987648006,"FDR":1,"overlapId":"84420"},{"geneSet":"GO:0016849","description":"phosphorus-oxygen lyase activity","link":"http://amigo.geneontology.org/amigo/term/GO:0016849","size":21,"overlap":1,"expect":0.00659392416987205,"enrichmentRatio":151.654761904762,"pValue":0.00657840987648006,"FDR":1,"overlapId":"84420"},{"geneSet":"GO:0019957","description":"C-C chemokine binding","link":"http://amigo.geneontology.org/amigo/term/GO:0019957","size":21,"overlap":1,"expect":0.00659392416987205,"enrichmentRatio":151.654761904762,"pValue":0.00657840987648006,"FDR":1,"overlapId":"301086"},{"geneSet":"GO:0016769","description":"transferase activity, transferring nitrogenous groups","link":"http://amigo.geneontology.org/amigo/term/GO:0016769","size":22,"overlap":1,"expect":0.00690792055891357,"enrichmentRatio":144.761363636364,"pValue":0.00689085572964077,"FDR":1,"overlapId":"64313"},{"geneSet":"GO:0016493","description":"C-C chemokine receptor activity","link":"http://amigo.geneontology.org/amigo/term/GO:0016493","size":22,"overlap":1,"expect":0.00690792055891357,"enrichmentRatio":144.761363636364,"pValue":0.00689085572964077,"FDR":1,"overlapId":"301086"},{"geneSet":"GO:0001637","description":"G protein-coupled chemoattractant receptor activity","link":"http://amigo.geneontology.org/amigo/term/GO:0001637","size":23,"overlap":1,"expect":0.0072219169479551,"enrichmentRatio":138.467391304348,"pValue":0.00720322787535743,"FDR":1,"overlapId":"301086"},{"geneSet":"GO:0004950","description":"chemokine receptor activity","link":"http://amigo.geneontology.org/amigo/term/GO:0004950","size":23,"overlap":1,"expect":0.0072219169479551,"enrichmentRatio":138.467391304348,"pValue":0.00720322787535743,"FDR":1,"overlapId":"301086"}];
var background = [{"geneSet":"GO:0019956","enrichmentRatio":132.697916666667,"FDR":1,"overlap":1},{"geneSet":"GO:0030170","enrichmentRatio":74.0639534883721,"FDR":1,"overlap":1},{"geneSet":"GO:0070279","enrichmentRatio":70.7722222222222,"FDR":1,"overlap":1},{"geneSet":"GO:0004896","enrichmentRatio":46.8345588235294,"FDR":1,"overlap":1},{"geneSet":"GO:0019955","enrichmentRatio":36.6063218390805,"FDR":1,"overlap":1},{"geneSet":"GO:0019842","enrichmentRatio":29.7640186915888,"FDR":1,"overlap":1},{"geneSet":"GO:0008528","enrichmentRatio":25.6834677419355,"FDR":1,"overlap":1},{"geneSet":"GO:0001653","enrichmentRatio":24.1268939393939,"FDR":1,"overlap":1},{"geneSet":"GO:0016829","enrichmentRatio":22.5868794326241,"FDR":1,"overlap":1},{"geneSet":"GO:0008144","enrichmentRatio":4.94526397515528,"FDR":1,"overlap":2},{"geneSet":"GO:0050662","enrichmentRatio":13.1059670781893,"FDR":1,"overlap":1},{"geneSet":"GO:0097367","enrichmentRatio":3.92936458975941,"FDR":1,"overlap":2},{"geneSet":"GO:0036094","enrichmentRatio":3.32610966057441,"FDR":1,"overlap":2},{"geneSet":"GO:0048037","enrichmentRatio":7.23806818181818,"FDR":1,"overlap":1},{"geneSet":"GO:0008289","enrichmentRatio":6.07776717557252,"FDR":1,"overlap":1},{"geneSet":"GO:0005524","enrichmentRatio":2.94611470860315,"FDR":1,"overlap":1},{"geneSet":"GO:0032559","enrichmentRatio":2.80842151675485,"FDR":1,"overlap":1},{"geneSet":"GO:0030554","enrichmentRatio":2.78630796150481,"FDR":1,"overlap":1},{"geneSet":"GO:0035639","enrichmentRatio":2.34690493736183,"FDR":1,"overlap":1},{"geneSet":"GO:0032555","enrichmentRatio":2.24594499294781,"FDR":1,"overlap":1},{"geneSet":"GO:0017076","enrichmentRatio":2.22865640307908,"FDR":1,"overlap":1},{"geneSet":"GO:0032553","enrichmentRatio":2.22088563458856,"FDR":1,"overlap":1},{"geneSet":"GO:0004930","enrichmentRatio":2.15768970189702,"FDR":1,"overlap":1},{"geneSet":"GO:0000166","enrichmentRatio":1.97810559006211,"FDR":1,"overlap":1},{"geneSet":"GO:1901265","enrichmentRatio":1.97810559006211,"FDR":1,"overlap":1},{"geneSet":"GO:0016740","enrichmentRatio":1.8366493656286,"FDR":1,"overlap":1},{"geneSet":"GO:0004888","enrichmentRatio":1.83453341013825,"FDR":1,"overlap":1},{"geneSet":"GO:0038023","enrichmentRatio":1.70581146223889,"FDR":1,"overlap":1},{"geneSet":"GO:0060089","enrichmentRatio":1.67883500263574,"FDR":1,"overlap":1}];
var geneTables = {"GO:0004016":[{"userId":"Adcy7","geneSymbol":"Adcy7","geneName":"adenylate cyclase 7","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=84420","entrezgene":"84420"}],"GO:0001530":[{"userId":"Tril","geneSymbol":"Tril","geneName":"TLR4 interactor with leucine-rich repeats","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=362364","entrezgene":"362364"}],"GO:0008483":[{"userId":"Oat","geneSymbol":"Oat","geneName":"ornithine aminotransferase","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=64313","entrezgene":"64313"}],"GO:0009975":[{"userId":"Adcy7","geneSymbol":"Adcy7","geneName":"adenylate cyclase 7","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=84420","entrezgene":"84420"}],"GO:0016849":[{"userId":"Adcy7","geneSymbol":"Adcy7","geneName":"adenylate cyclase 7","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=84420","entrezgene":"84420"}],"GO:0019957":[{"userId":"Xcr1","geneSymbol":"Xcr1","geneName":"X-C motif chemokine receptor 1","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=301086","entrezgene":"301086"}],"GO:0016769":[{"userId":"Oat","geneSymbol":"Oat","geneName":"ornithine aminotransferase","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=64313","entrezgene":"64313"}],"GO:0016493":[{"userId":"Xcr1","geneSymbol":"Xcr1","geneName":"X-C motif chemokine receptor 1","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=301086","entrezgene":"301086"}],"GO:0001637":[{"userId":"Xcr1","geneSymbol":"Xcr1","geneName":"X-C motif chemokine receptor 1","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=301086","entrezgene":"301086"}],"GO:0004950":[{"userId":"Xcr1","geneSymbol":"Xcr1","geneName":"X-C motif chemokine receptor 1","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=301086","entrezgene":"301086"}]};
var clusters = {"ap":{"clusters":[["GO:0004016","GO:0009975","GO:0016849"],["GO:0001530"],["GO:0008483","GO:0016769"],["GO:0019957","GO:0016493","GO:0001637","GO:0004950"]],"representatives":["GO:0004016","GO:0001530","GO:0008483","GO:0019957"]},"wsc":{"representatives":["GO:0004016","GO:0001530","GO:0008483","GO:0019957"],"coverage":[1]}};

var dag = {"geneontology_Molecular_Function":[{"data":{"source":"GO:0009975","target":"GO:0004016"}},{"data":{"source":"GO:0016849","target":"GO:0004016"}},{"data":{"source":"GO:0008289","target":"GO:0001530"}},{"data":{"source":"GO:0097367","target":"GO:0001530"}},{"data":{"source":"GO:0016769","target":"GO:0008483"}},{"data":{"source":"GO:0003824","target":"GO:0009975"}},{"data":{"source":"GO:0016829","target":"GO:0016849"}},{"data":{"source":"GO:0019956","target":"GO:0019957"}},{"data":{"source":"GO:0016740","target":"GO:0016769"}},{"data":{"source":"GO:0004950","target":"GO:0016493"}},{"data":{"source":"GO:0008528","target":"GO:0001637"}},{"data":{"source":"GO:0001637","target":"GO:0004950"}},{"data":{"source":"GO:0004896","target":"GO:0004950"}},{"data":{"source":"GO:0005488","target":"GO:0008289"}},{"data":{"source":"GO:0005488","target":"GO:0097367"}},{"data":{"source":"GO:0003674","target":"GO:0003824"}},{"data":{"source":"GO:0003824","target":"GO:0016829"}},{"data":{"source":"GO:0019955","target":"GO:0019956"}},{"data":{"source":"GO:0003824","target":"GO:0016740"}},{"data":{"source":"GO:0001653","target":"GO:0008528"}},{"data":{"source":"GO:0004930","target":"GO:0008528"}},{"data":{"source":"GO:0004888","target":"GO:0004896"}},{"data":{"source":"GO:0003674","target":"GO:0005488"}},{"data":{"source":"GO:0005515","target":"GO:0019955"}},{"data":{"source":"GO:0038023","target":"GO:0001653"}},{"data":{"source":"GO:0004888","target":"GO:0004930"}},{"data":{"source":"GO:0038023","target":"GO:0004888"}},{"data":{"source":"GO:0005488","target":"GO:0005515"}},{"data":{"source":"GO:0060089","target":"GO:0038023"}},{"data":{"source":"GO:0003674","target":"GO:0060089"}},{"data":{"id":"GO:0004016","name":"adenylate cyclase activity","color":"#FFFFFF"}},{"data":{"id":"GO:0001530","name":"lipopolysaccharide binding","color":"#FFFFFF"}},{"data":{"id":"GO:0008483","name":"transaminase activity","color":"#FFFFFF"}},{"data":{"id":"GO:0009975","name":"cyclase activity","color":"#FFFFFF"}},{"data":{"id":"GO:0016849","name":"phosphorus-oxygen lyase activity","color":"#FFFFFF"}},{"data":{"id":"GO:0019957","name":"C-C chemokine binding","color":"#FFFFFF"}},{"data":{"id":"GO:0016769","name":"transferase activity, transferring nitrogenous groups","color":"#FFFFFF"}},{"data":{"id":"GO:0016493","name":"C-C chemokine receptor activity","color":"#FFFFFF"}},{"data":{"id":"GO:0001637","name":"G protein-coupled chemoattractant receptor activity","color":"#FFFFFF"}},{"data":{"id":"GO:0004950","name":"chemokine receptor activity","color":"#FFFFFF"}},{"data":{"id":"GO:0008289","name":"lipid binding","color":"white"}},{"data":{"id":"GO:0097367","name":"carbohydrate derivative binding","color":"white"}},{"data":{"id":"GO:0003824","name":"catalytic activity","color":"white"}},{"data":{"id":"GO:0016829","name":"lyase activity","color":"white"}},{"data":{"id":"GO:0019956","name":"chemokine binding","color":"white"}},{"data":{"id":"GO:0016740","name":"transferase activity","color":"white"}},{"data":{"id":"GO:0008528","name":"G protein-coupled peptide receptor activity","color":"white"}},{"data":{"id":"GO:0004896","name":"cytokine receptor activity","color":"white"}},{"data":{"id":"GO:0005488","name":"binding","color":"white"}},{"data":{"id":"GO:0003674","name":"molecular_function","color":"white"}},{"data":{"id":"GO:0019955","name":"cytokine binding","color":"white"}},{"data":{"id":"GO:0001653","name":"peptide receptor activity","color":"white"}},{"data":{"id":"GO:0004930","name":"G protein-coupled receptor activity","color":"white"}},{"data":{"id":"GO:0004888","name":"transmembrane signaling receptor activity","color":"white"}},{"data":{"id":"GO:0005515","name":"protein binding","color":"white"}},{"data":{"id":"GO:0038023","name":"signaling receptor activity","color":"white"}},{"data":{"id":"GO:0060089","name":"molecular transducer activity","color":"white"}}]};
wgLib({
	enrichDatabase: "geneontology_Molecular_Function",
	organism: "rnorvegicus",
	hostName: "https://www.webgestalt.org",
	enrichment: enrichment,
	background: background,
	geneTables: geneTables,
	clusters: clusters,
	
	enrichMethod: "ora",
	mappedInputSize: 4,
	standardId: "entrezgene",
	dag: dag,
	volcano: {
		labelTypes: ["geneSet", "description"],
				dimensions: { width: 960, height: 500 },
		autoPlaceLabels: true,
		y: "FDR",


		x: "enrichmentRatio",
		xLabel: "log2 of enrichment ratio",
		radius: "overlap",
		color: "overlap",
			tooltipContent: ["geneSet", "description", "enrichmentRatio", "FDR", "pValue", "overlap"]
					
	},
	bar: {
		dimensions: { width: 960, height: 300 },
		labelWidth: 200,
		tooltipContent: ["geneSet", "description", "enrichmentRatio", "FDR", "pValue", "overlap"],
		
		valueKey: "enrichmentRatio",
		xLabel: "Enrichment ratio"
	}
})


</script>
<footer class="footer has-text-centered">
<p>WebGestalt is currently developed and maintained by Yuxing Liao, Suhas Vasaikar, Zhiao Shi and Bing Zhang at the <a target="_new" href=http://www.zhang-lab.org> Zhang Lab</a>. Other people who have made significant contribution to the project include Jing Wang, Dexter Duncan, Stefan Kirov and Jay Snoddy.</p>
<p><b>Funding credits:</b> NIH/NCI (U24 CA210954); Leidos (15X038); CPRIT (RR160027); NIH/NIAAA (U01 AA016662, U01 AA013512); NIH/NIDA (P01 DA015027); NIH/NIMH (P50 MH078028, P50 MH096972); NIH/NCI (U24 CA159988); NIH/NIGMS (R01 GM088822).
</p>
</footer></body>
</html>