<!DOCTYPE html>
<html>
<head>
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta charset="UTF-8">
<title>WebGestalt (WEB-based GEne SeT AnaLysis Toolkit)</title>
<link rel="icon" type="image/png" href="https://s3-us-west-2.amazonaws.com/webgestalt/favicon.png">
<script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');
ga('create', 'UA-42098540-1', 'auto');
ga('send', 'pageview');
</script>
<script src="https://polyfill.io/v3/polyfill.min.js?flags=gated&features=es2015%2Ces2016%2Cfetch"></script>
<script src="https://www.webgestalt.org/assets/0.4/build/vendor.min.js"></script>
	
	<script>window.cytoscape = null;</script>
	<link rel="stylesheet" href="https://www.webgestalt.org/assets/0.4/build/vendor.css">
	<link rel="stylesheet" href="https://www.webgestalt.org/assets/0.4/build/wg.css">
<link rel="stylesheet" href="https://cdn.materialdesignicons.com/2.5.94/css/materialdesignicons.min.css">
</head>
<body>
<header>
	<div id="header-img"><a href="https://www.webgestalt.org"><img border="0" src="https://www.webgestalt.org/images/gsa.png"></a></div>
	<div id="header-title">
		<p id="header-title-text">WEB-based GEne SeT AnaLysis Toolkit</p>
		<p id="header-subtitle-text">Translating gene lists into biological insights...</p>
	</div>
</header>
<hr><main>
<div id="app">
<div class="loading-preinit" v-cloak><div class="loading-icon"></div></div>
<div class="wrapper" v-cloak>
<div class="level">
	<div class="level-left">
		<div class="level-item">
			<h4 class="title is-4 has-text-primary">Summary</h4>
		</div>
	</div>
	<div class="level-right">
		<div class="level-item">
			<a class="button is-text" href="Project_wg_result1698902118.zip" target="_blank">Result Download</a>
		</div>
	</div>
</div>

<section id="summary" class="summary">
<div class="card" v-bind:class="{active: summaryShown}" >
	<div class="card-header" v-on:click="summaryShown=!summaryShown">
		<p class="card-header-title">Job summary</p>
		<a href="#" class="card-header-icon">
			<b-icon :icon="summaryShown ? 'chevron-up' : 'chevron-down'"></b-icon>
		</a>
	</div>
	<div class="card-content content" v-show="summaryShown">
		<ul id="summary-list"> 
		<li><b>Enrichment method:</b> ORA</li>
		<li><b>Organism: </b>rnorvegicus</li>
		<li>
				<b>Enrichment Categories: </b>pathway_KEGG
				<a class="has-text-info" href="https://www.webgestalt.org/api/geneset?organism=rnorvegicus&database=pathway_KEGG&fileType=gmt">
					<b-icon icon="download" size="is-small"></b-icon>
				</a>
					</li>
		
			<li><b>Interesting list: </b>textAreaUpload_1698902118.txt. <b>ID type: </b>genesymbol</li>
						<li>The interesting list contains <b>27</b> user IDs in which <b>27</b> user IDs are unambiguously mapped to <b>27</b> unique entrezgene IDs and <b>0</b> user IDs can not be mapped to any entrezgene ID.</li>
			<li>The GO Slim summary are based upon the <b>27</b> unique entrezgene IDs.</li>
		

		<li>Among <b>27</b> unique entrezgene IDs, <b>10</b> IDs are annotated to the selected functional categories and also in the reference list, which are used for the enrichment analysis.</li>
		
		<li><b>Reference list: </b> all mapped entrezgene IDs from the selected platform genome
			<a class="has-text-info" href="https://www.webgestalt.org/api/reference?organism=rnorvegicus&referenceSet=genome">
				<b-icon icon="download" size="is-small"></b-icon>
			</a>
		</li>
	
	<li>The reference list can be mapped to <b>47958</b> entrezgene IDs and <b> 8557</b> IDs are annotated to the selected functional categories that are used as the reference for the enrichment analysis.</li>
</ul> 

<p><b>Parameters for the enrichment analysis:</b></p>
<ul><li><b>Minimum number of IDs in the category: </b>5</li>
<li><b>Maximum number of IDs in the category: </b>2000</li>
<li><b>FDR Method: </b>BH</li>

<li><b>Significance Level:</b> Top 10</li>
</ul>
	
	<p>Based on the above parameters, <b>10</b> categories are identified as enriched categories and all are shown in this report.</p>
	

	</div>
</div>
</section>
<section id="go-summary" class="summary">
<div class="card" v-bind:class="{active: goShown}" >
	<div class="card-header" v-on:click="goShown=!goShown">
		<p class="card-header-title">GO Slim summary for the user uploaded IDs</p>
		<a href="#" class="card-header-icon">
			<b-icon :icon="goShown ? 'chevron-up' : 'chevron-down'"></b-icon>
		</a>
	</div>
	<div v-show="goShown">
		<div class="card-content">
			<p>Each Biological Process, Cellular Component and Molecular Function category is represented by a red, blue and green bar, repectively.</p>
			<p>The height of the bar represents the number of IDs in the user list and also in the category.</p>
		</div>
		<div class="card-image">
			<img src="goslim_summary_wg_result1698902118.png" width="100%" height="100%" alt="Bar charts of enriched GO Slim terms"/>
		</div>
	</div>
</div>
</section> <h4 class="title is-4 has-text-primary">Enrichment Results</h4>
<section id="enrich-result">
	<p class="select-redundancy">
		<b-tooltip label="Enriched gene sets are post-processed by two methods to reduce redundancy." type="is-dark" animated multilined dashed>
			Redundancy reduction:
		</b-tooltip>
		<b-radio v-model="redundancy" native-value="all">All</b-radio>
		<b-radio v-model="redundancy" native-value="ap">
			<b-tooltip label="R package apcluster is used to cluster gene sets." type="is-dark" animated multilined dashed>
				Affinity propagation
			</b-tooltip>
		</b-radio>
		
		<b-radio v-model="redundancy" native-value="wsc">
			<b-tooltip label="Find top gene sets while maximizing gene coverage." type="is-dark" animated multilined dashed>
				Weighted set cover
			</b-tooltip>
		</b-radio>
	</p>
	<b-tabs v-model="activeTab" type="is-toggle">
		<b-tab-item label="Table">
			<wg-result-table
				:data="enrichmentData"
				:method="enrichMethod"
			></wg-result-table>
		</b-tab-item>
		<b-tab-item label="Bar chart">
		<div class="vertical-center-container">
			<wg-bar-plot
				:dimensions="bar.dimensions"
				:label-width="bar.labelWidth"
				:data="enrichmentData"
				:value-key="bar.valueKey"
				:x-label="bar.xLabel">
			</wg-bar-plot>
		</div>
		</b-tab-item>
		<b-tab-item label="Volcano plot">
		<div class="vertical-center-container">
			<wg-volcano-plot
				:data="volcanoData"
				:x="volcano.x"
				:y="volcano.y"
				:dimensions="volcano.dimensions"
				:x-label="volcano.xLabel"
				:label-types="volcano.labelTypes"
				:x-transform="volcano.xTransform"
				:radius="volcano.radius"
				:color="volcano.color"
				:tooltip-content="volcano.tooltipContent"
				:show-all-labels="false"
				:auto-place-labels="true"
			></wg-volcano-plot>
		</div>
		</b-tab-item>
	</b-tabs>
	<wg-gene-table
		:data="geneTables"
		:info="enrichment"
		:standard-id="standardId"
		:method="enrichMethod"
		:mapped-input-size="mappedInputSize"
		:organism="organism"
		:host-name="hostName"
		:sets-with-net="false"
	></wg-gene-table>
</section> 

</div>
</div>
</main>
	<script type="text/javascript" src="https://www.webgestalt.org/assets/0.4/build/wg.min.js"></script>
<script type="text/javascript">
var enrichment = [{"geneSet":"rno00512","description":"Mucin type O-glycan biosynthesis","link":"http://www.kegg.jp/kegg-bin/show_pathway?rno00512+64043","size":28,"overlap":1,"expect":0.032721748276265,"enrichmentRatio":30.5607142857143,"pValue":0.0322608277566029,"FDR":1,"overlapId":"64043"},{"geneSet":"rno04215","description":"Apoptosis","link":"http://www.kegg.jp/kegg-bin/show_pathway?rno04215+492821","size":34,"overlap":1,"expect":0.0397335514783218,"enrichmentRatio":25.1676470588235,"pValue":0.0390507617165716,"FDR":1,"overlapId":"492821"},{"geneSet":"rno00860","description":"Porphyrin and chlorophyll metabolism","link":"http://www.kegg.jp/kegg-bin/show_pathway?rno00860+24268","size":41,"overlap":1,"expect":0.0479139885473881,"enrichmentRatio":20.8707317073171,"pValue":0.0469181395877557,"FDR":1,"overlapId":"24268"},{"geneSet":"rno04216","description":"Ferroptosis","link":"http://www.kegg.jp/kegg-bin/show_pathway?rno04216+24268","size":41,"overlap":1,"expect":0.0479139885473881,"enrichmentRatio":20.8707317073171,"pValue":0.0469181395877557,"FDR":1,"overlapId":"24268"},{"geneSet":"rno05144","description":"Malaria","link":"http://www.kegg.jp/kegg-bin/show_pathway?rno05144+24934","size":60,"overlap":1,"expect":0.070118032020568,"enrichmentRatio":14.2616666666667,"pValue":0.0679810912714387,"FDR":1,"overlapId":"24934"},{"geneSet":"rno04115","description":"p53 signaling pathway","link":"http://www.kegg.jp/kegg-bin/show_pathway?rno04115+492821","size":74,"overlap":1,"expect":0.0864789061587005,"enrichmentRatio":11.5635135135135,"pValue":0.0832320708553054,"FDR":1,"overlapId":"492821"},{"geneSet":"rno04920","description":"Adipocytokine signaling pathway","link":"http://www.kegg.jp/kegg-bin/show_pathway?rno04920+24536","size":74,"overlap":1,"expect":0.0864789061587005,"enrichmentRatio":11.5635135135135,"pValue":0.0832320708553054,"FDR":1,"overlapId":"24536"},{"geneSet":"rno01524","description":"Platinum drug resistance","link":"http://www.kegg.jp/kegg-bin/show_pathway?rno01524+492821","size":82,"overlap":1,"expect":0.0958279770947762,"enrichmentRatio":10.4353658536585,"pValue":0.0918457188731798,"FDR":1,"overlapId":"492821"},{"geneSet":"rno04911","description":"Insulin secretion","link":"http://www.kegg.jp/kegg-bin/show_pathway?rno04911+252857","size":86,"overlap":1,"expect":0.100502512562814,"enrichmentRatio":9.95,"pValue":0.0961251685650668,"FDR":1,"overlapId":"252857"},{"geneSet":"rno04350","description":"TGF-beta signaling pathway","link":"http://www.kegg.jp/kegg-bin/show_pathway?rno04350+29139","size":87,"overlap":1,"expect":0.101671146429824,"enrichmentRatio":9.83563218390804,"pValue":0.0971921911496907,"FDR":1,"overlapId":"29139"}];
var background = [{"geneSet":"rno04540","enrichmentRatio":9.72386363636364,"FDR":1,"overlap":1},{"geneSet":"rno05210","enrichmentRatio":9.72386363636364,"FDR":1,"overlap":1},{"geneSet":"rno04974","enrichmentRatio":9.1031914893617,"FDR":1,"overlap":1},{"geneSet":"rno04650","enrichmentRatio":8.64343434343434,"FDR":1,"overlap":1},{"geneSet":"rno04928","enrichmentRatio":8.07264150943396,"FDR":1,"overlap":1},{"geneSet":"rno04670","enrichmentRatio":7.44086956521739,"FDR":1,"overlap":1},{"geneSet":"rno04152","enrichmentRatio":6.73779527559055,"FDR":1,"overlap":1},{"geneSet":"rno04210","enrichmentRatio":6.06879432624113,"FDR":1,"overlap":1},{"geneSet":"rno04261","enrichmentRatio":5.94236111111111,"FDR":1,"overlap":1},{"geneSet":"rno04072","enrichmentRatio":5.74295302013423,"FDR":1,"overlap":1},{"geneSet":"rno04630","enrichmentRatio":5.52064516129032,"FDR":1,"overlap":1},{"geneSet":"rno04932","enrichmentRatio":5.38176100628931,"FDR":1,"overlap":1},{"geneSet":"rno05202","enrichmentRatio":4.65054347826087,"FDR":1,"overlap":1},{"geneSet":"rno04024","enrichmentRatio":4.34365482233502,"FDR":1,"overlap":1},{"geneSet":"rno04145","enrichmentRatio":4.32171717171717,"FDR":1,"overlap":1},{"geneSet":"rno05205","enrichmentRatio":4.15388349514563,"FDR":1,"overlap":1},{"geneSet":"rno04015","enrichmentRatio":4.01737089201878,"FDR":1,"overlap":1},{"geneSet":"rno05203","enrichmentRatio":3.7204347826087,"FDR":1,"overlap":1},{"geneSet":"rno04060","enrichmentRatio":3.18104089219331,"FDR":1,"overlap":1},{"geneSet":"rno04080","enrichmentRatio":2.87147651006711,"FDR":1,"overlap":1},{"geneSet":"rno05200","enrichmentRatio":1.60544090056285,"FDR":1,"overlap":1},{"geneSet":"rno01100","enrichmentRatio":0.620072463768116,"FDR":1,"overlap":1}];
var geneTables = {"rno00512":[{"userId":"Gcnt1","geneSymbol":"Gcnt1","geneName":"glucosaminyl (N-acetyl) transferase 1","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=64043","entrezgene":"64043"}],"rno04215":[{"userId":"Pmaip1","geneSymbol":"Pmaip1","geneName":"phorbol-12-myristate-13-acetate-induced protein 1","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=492821","entrezgene":"492821"}],"rno00860":[{"userId":"Cp","geneSymbol":"Cp","geneName":"ceruloplasmin","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=24268","entrezgene":"24268"}],"rno04216":[{"userId":"Cp","geneSymbol":"Cp","geneName":"ceruloplasmin","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=24268","entrezgene":"24268"}],"rno05144":[{"userId":"Klrk1","geneSymbol":"Klrk1","geneName":"killer cell lectin like receptor K1","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=24934","entrezgene":"24934"}],"rno04115":[{"userId":"Pmaip1","geneSymbol":"Pmaip1","geneName":"phorbol-12-myristate-13-acetate-induced protein 1","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=492821","entrezgene":"492821"}],"rno04920":[{"userId":"Lepr","geneSymbol":"Lepr","geneName":"leptin receptor","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=24536","entrezgene":"24536"}],"rno01524":[{"userId":"Pmaip1","geneSymbol":"Pmaip1","geneName":"phorbol-12-myristate-13-acetate-induced protein 1","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=492821","entrezgene":"492821"}],"rno04911":[{"userId":"Rapgef4","geneSymbol":"Rapgef4","geneName":"Rap guanine nucleotide exchange factor 4","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=252857","entrezgene":"252857"}],"rno04350":[{"userId":"Dcn","geneSymbol":"Dcn","geneName":"decorin","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=29139","entrezgene":"29139"}]};
var clusters = {"ap":{"clusters":[["rno00512"],["rno04215","rno04115","rno01524"],["rno04216","rno00860"],["rno05144"],["rno04920"],["rno04911"],["rno04350"]],"representatives":["rno00512","rno04215","rno04216","rno05144","rno04920","rno04911","rno04350"]},"wsc":{"representatives":["rno00512","rno04215","rno00860","rno05144","rno04920","rno04911","rno04350"],"coverage":[1]}};

wgLib({
	enrichDatabase: "pathway_KEGG",
	organism: "rnorvegicus",
	hostName: "https://www.webgestalt.org",
	enrichment: enrichment,
	background: background,
	geneTables: geneTables,
	clusters: clusters,
	
	enrichMethod: "ora",
	mappedInputSize: 10,
	standardId: "entrezgene",
	volcano: {
		labelTypes: ["geneSet", "description"],
				dimensions: { width: 960, height: 500 },
		autoPlaceLabels: true,
		y: "FDR",


		x: "enrichmentRatio",
		xLabel: "log2 of enrichment ratio",
		radius: "overlap",
		color: "overlap",
			tooltipContent: ["geneSet", "description", "enrichmentRatio", "FDR", "pValue", "overlap"]
					
	},
	bar: {
		dimensions: { width: 960, height: 300 },
		labelWidth: 200,
		tooltipContent: ["geneSet", "description", "enrichmentRatio", "FDR", "pValue", "overlap"],
		
		valueKey: "enrichmentRatio",
		xLabel: "Enrichment ratio"
	}
})


</script>
<footer class="footer has-text-centered">
<p>WebGestalt is currently developed and maintained by Yuxing Liao, Suhas Vasaikar, Zhiao Shi and Bing Zhang at the <a target="_new" href=http://www.zhang-lab.org> Zhang Lab</a>. Other people who have made significant contribution to the project include Jing Wang, Dexter Duncan, Stefan Kirov and Jay Snoddy.</p>
<p><b>Funding credits:</b> NIH/NCI (U24 CA210954); Leidos (15X038); CPRIT (RR160027); NIH/NIAAA (U01 AA016662, U01 AA013512); NIH/NIDA (P01 DA015027); NIH/NIMH (P50 MH078028, P50 MH096972); NIH/NCI (U24 CA159988); NIH/NIGMS (R01 GM088822).
</p>
</footer></body>
</html>