<!DOCTYPE html>
<html>
<head>
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta charset="UTF-8">
<title>WebGestalt (WEB-based GEne SeT AnaLysis Toolkit)</title>
<link rel="icon" type="image/png" href="https://s3-us-west-2.amazonaws.com/webgestalt/favicon.png">

<script src="https://polyfill.io/v3/polyfill.min.js?flags=gated&features=es2015%2Ces2016%2Cfetch"></script>
<script src="https://2024.webgestalt.org/assets/1.0/build/vendor.min.js"></script>
	<script>window.cytoscape = null;</script>
	<link rel="stylesheet" href="https://2024.webgestalt.org/assets/1.0/build/vendor.css">
	<link rel="stylesheet" href="https://2024.webgestalt.org/assets/1.0/build/wg.css">
<link rel="stylesheet" href="https://cdn.materialdesignicons.com/2.5.94/css/materialdesignicons.min.css">
</head>
<body>
	<script type="text/javascript" src="https://www.termsfeed.com/public/cookie-consent/4.1.0/cookie-consent.js" charset="UTF-8"></script>
<script type="text/javascript" charset="UTF-8">
document.addEventListener('DOMContentLoaded', function () {
cookieconsent.run({"notice_banner_type":"simple","consent_type":"express","palette":"light","language":"en","page_load_consent_levels":["strictly-necessary"],"notice_banner_reject_button_hide":false,"preferences_center_close_button_hide":false,"page_refresh_confirmation_buttons":false});
});
</script>

<!-- Google Analytics -->
<script type="text/plain" data-cookie-consent="tracking" async src="https://www.googletagmanager.com/gtag/js?id=G-THG9Q3WPEJ"></script>
<script type="text/plain" data-cookie-consent="tracking">
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-THG9Q3WPEJ');
</script>
<!-- end of Google Analytics-->
	
	<noscript>Free cookie consent management tool by <a href="https://www.termsfeed.com/">TermsFeed</a></noscript>
	<!-- End Cookie Consent by TermsFeed https://www.TermsFeed.com -->
<header>
	<div id="header-img"><a href="https://2024.webgestalt.org"><img border="0" src="https://2024.webgestalt.org/images/gsa.png"></a></div>
	<div id="header-title">
		<p id="header-title-text">WEB-based GEne SeT AnaLysis Toolkit</p>
		<p id="header-subtitle-text">Translating gene lists into biological insights...</p>
	</div>
</header>
<hr><main>
<div id="app">
<div class="loading-preinit" v-cloak><div class="loading-icon"></div></div>
<div class="wrapper" v-cloak>
<div class="level">
	<div class="level-left">
		<div class="level-item">
			<h4 class="title is-4 has-text-primary">Summary</h4>
		</div>
	</div>
	<div class="level-right">
		<div class="level-item">
			<a class="button is-text" href="Project_wg_result1720233427.zip" target="_blank">Result Download</a>
		</div>
	</div>
</div>

<section id="summary" class="summary">
<div class="card" v-bind:class="{active: summaryShown}" >
	<div class="card-header" v-on:click="summaryShown=!summaryShown">
		<p class="card-header-title">Job summary</p>
		<a href="#" class="card-header-icon">
			<b-icon :icon="summaryShown ? 'chevron-up' : 'chevron-down'"></b-icon>
		</a>
	</div>
	<div class="card-content content" v-show="summaryShown">
		<ul id="summary-list"> 
		<li><b>Enrichment method:</b> ORA</li>
		<li><b>Organism: </b>hsapiens</li>
		<li>
				<b>Enrichment Categories: </b>pathway_WikiPathways
				<a class="has-text-info" href="https://2024.webgestalt.org/api/geneset?organism=hsapiens&database=pathway_WikiPathways&fileType=gmt">
					<b-icon icon="download" size="is-small"></b-icon>
				</a>
		</li>

			<li><b>Interesting list: </b>textAreaUpload_1720233427.txt. <b>ID type: </b>genesymbol</li>
			<li>The interesting list contains <b>40</b> user IDs in which <b>36</b> user IDs are unambiguously mapped to <b>36</b> unique entrezgene IDs and <b>4</b> user IDs can not be mapped to any entrezgene ID.</li>
			<li>The GO Slim summary are based upon the <b>36</b> unique entrezgene IDs.</li>
		<li>Among <b>36</b> unique entrezgene IDs, <b>17</b> IDs are annotated to the selected functional categories and also in the reference list, which are used for the enrichment analysis.</li>
		<li><b>Reference list: </b> all mapped entrezgene IDs from the selected platform genome
			<a class="has-text-info" href="https://2024.webgestalt.org/api/reference?organism=hsapiens&referenceSet=genome">
				<b-icon icon="download" size="is-small"></b-icon>
			</a>
		</li>
	<li>The reference list can be mapped to <b>193394</b> entrezgene IDs and <b> 8382</b> IDs are annotated to the selected functional categories that are used as the reference for the enrichment analysis.</li>
</ul> 

<p><b>Parameters for the enrichment analysis:</b></p>
<ul><li><b>Minimum number of IDs in the category: </b>5</li>
<li><b>Maximum number of IDs in the category: </b>2000</li>
<li><b>FDR Method: </b>BH</li>
<li><b>Significance Level:</b> Top 10</li>
</ul>
	<p>Based on the above parameters, <b>10</b> categories are identified as enriched categories and all are shown in this report.</p>

	</div>
</div>
</section>
<section id="go-summary" class="summary">
<div class="card" v-bind:class="{active: goShown}" >
	<div class="card-header" v-on:click="goShown=!goShown">
		<p class="card-header-title">GO Slim summary for the user uploaded IDs</p>
		<a href="#" class="card-header-icon">
			<b-icon :icon="goShown ? 'chevron-up' : 'chevron-down'"></b-icon>
		</a>
	</div>
	<div v-show="goShown">
		<div class="card-content">
			<p>Each Biological Process, Cellular Component and Molecular Function category is represented by a red, blue and green bar, repectively.</p>
			<p>The height of the bar represents the number of IDs in the user list and also in the category.</p>
		</div>
		<div class="card-image">
			<img src="goslim_summary_wg_result1720233427.png" width="100%" height="100%" alt="Bar charts of enriched GO Slim terms"/>
		</div>
	</div>
</div>
</section> <h4 class="title is-4 has-text-primary">Enrichment Results</h4>
<section id="enrich-result">
	<p class="select-redundancy">
		<b-tooltip label="Enriched analyte sets are post-processed by two methods to reduce redundancy." type="is-dark" animated multilined dashed>
			Redundancy reduction:
		</b-tooltip>
		<b-radio v-model="redundancy" native-value="all">None</b-radio>
		<b-radio v-model="redundancy" native-value="wsc">
			<b-tooltip label="Find top analyte sets while maximizing analyte coverage." type="is-dark" animated multilined dashed>
				Weighted set cover
			</b-tooltip>
		</b-radio>
	</p>
	<b-tabs v-model="activeTab" type="is-toggle">
		<b-tab-item label="Table">
			<wg-result-table
				:data="enrichmentData"
				:method="enrichMethod"
			></wg-result-table>
		</b-tab-item>
		<b-tab-item label="Bar chart">
		<div class="vertical-center-container">
			<wg-bar-plot
				:dimensions="bar.dimensions"
				:label-width="bar.labelWidth"
				:data="enrichmentData"
				:value-key="bar.valueKey"
				:x-label="bar.xLabel">
			</wg-bar-plot>
		</div>
		</b-tab-item>
		<b-tab-item label="Volcano plot">
		<div class="vertical-center-container">
			<wg-volcano-plot
				:data="volcanoData"
				:x="volcano.x"
				:y="volcano.y"
				:dimensions="volcano.dimensions"
				:x-label="volcano.xLabel"
				:label-types="volcano.labelTypes"
				:x-transform="volcano.xTransform"
				:radius="volcano.radius"
				:color="volcano.color"
				:tooltip-content="volcano.tooltipContent"
				:show-all-labels="false"
				:auto-place-labels="true"
			></wg-volcano-plot>
		</div>
		</b-tab-item>
	</b-tabs>
	<wg-gene-table
		:data="geneTables"
		:info="enrichment"
		:standard-id="standardId"
		:method="enrichMethod"
		:mapped-input-size="mappedInputSize"
		:organism="organism"
		:host-name="hostName"
		:sets-with-net="false"
	></wg-gene-table>
</section> 

</div>
</div>
</main>
	<script type="text/javascript" src="https://2024.webgestalt.org/assets/1.0/build/wg.min.js"></script>
<script type="text/javascript">
var enrichment = [{"geneSet":"WP80","description":"Nucleotide GPCRs","link":"http://www.wikipathways.org/wpi/PathwayWidget.php?id=WP80&label[]=P2RY6&colors=steelblue","size":11,"overlap":1,"expect":0.022309711286089239,"enrichmentRatio":44.823529411764703,"pValue":0.022097895901937885,"FDR":1,"overlapId":"5031"},{"geneSet":"WP4584","description":"Biomarkers for pyrimidine metabolism disorders","link":"http://www.wikipathways.org/wpi/PathwayWidget.php?id=WP4584&label[]=UPB1&colors=steelblue","size":15,"overlap":1,"expect":0.03042233357193987,"enrichmentRatio":32.870588235294122,"pValue":0.03001892056170654,"FDR":1,"overlapId":"51733"},{"geneSet":"WP3664","description":"Regulation of Wnt B catenin signaling by small molecule compounds","link":"http://www.wikipathways.org/wpi/PathwayWidget.php?id=WP3664&label[]=SFRP4&colors=steelblue","size":17,"overlap":1,"expect":0.034478644714865191,"enrichmentRatio":29.003460207612456,"pValue":0.033956750147347003,"FDR":1,"overlapId":"6424"},{"geneSet":"WP5172","description":"Iron metabolism disorders","link":"http://www.wikipathways.org/wpi/PathwayWidget.php?id=WP5172&label[]=CP&colors=steelblue","size":17,"overlap":1,"expect":0.034478644714865191,"enrichmentRatio":29.003460207612456,"pValue":0.033956750147347003,"FDR":1,"overlapId":"1356"},{"geneSet":"WP4225","description":"Pyrimidine metabolism and related diseases","link":"http://www.wikipathways.org/wpi/PathwayWidget.php?id=WP4225&label[]=UPB1&colors=steelblue","size":20,"overlap":1,"expect":0.04056311142925316,"enrichmentRatio":24.652941176470588,"pValue":0.039835293360595457,"FDR":1,"overlapId":"51733"},{"geneSet":"WP4900","description":"Purinergic signaling","link":"http://www.wikipathways.org/wpi/PathwayWidget.php?id=WP4900&label[]=P2RY6&colors=steelblue","size":33,"overlap":1,"expect":0.066929133858267723,"enrichmentRatio":14.941176470588234,"pValue":0.064922108552098221,"FDR":1,"overlapId":"5031"},{"geneSet":"WP1601","description":"Fluoropyrimidine activity","link":"http://www.wikipathways.org/wpi/PathwayWidget.php?id=WP1601&label[]=UPB1&colors=steelblue","size":34,"overlap":1,"expect":0.068957289429730381,"enrichmentRatio":14.501730103806228,"pValue":0.066826087970489598,"FDR":1,"overlapId":"51733"},{"geneSet":"WP500","description":"Glycogen synthesis and degradation","link":"http://www.wikipathways.org/wpi/PathwayWidget.php?id=WP500&label[]=PYGM&colors=steelblue","size":40,"overlap":1,"expect":0.081126222858506319,"enrichmentRatio":12.326470588235294,"pValue":0.078173560037344159,"FDR":1,"overlapId":"5837"},{"geneSet":"WP716","description":"Vitamin A and carotenoid metabolism","link":"http://www.wikipathways.org/wpi/PathwayWidget.php?id=WP716&label[]=DHRS3&colors=steelblue","size":43,"overlap":1,"expect":0.087210689572894295,"enrichmentRatio":11.466484268125855,"pValue":0.083798472555588269,"FDR":1,"overlapId":"9249"},{"geneSet":"WP4577","description":"Neurodegeneration with brain iron accumulation NBIA subtypes pathway","link":"http://www.wikipathways.org/wpi/PathwayWidget.php?id=WP4577&label[]=CP&colors=steelblue","size":44,"overlap":1,"expect":0.089238845144356954,"enrichmentRatio":11.205882352941176,"pValue":0.085666253580255716,"FDR":1,"overlapId":"1356"}];
var background = [{"geneSet":"WP5078","enrichmentRatio":10.718670076726342,"FDR":1,"overlap":1},{"geneSet":"WP5346","enrichmentRatio":8.9647058823529413,"FDR":1,"overlap":1},{"geneSet":"WP558","enrichmentRatio":8.5010141987829613,"FDR":1,"overlap":1},{"geneSet":"WP5222","enrichmentRatio":7.8263305322128849,"FDR":1,"overlap":1},{"geneSet":"WP4313","enrichmentRatio":7.5855203619909499,"FDR":1,"overlap":1},{"geneSet":"WP2880","enrichmentRatio":7.0436974789915974,"FDR":1,"overlap":1},{"geneSet":"WP4336","enrichmentRatio":5.539986781229346,"FDR":1,"overlap":1},{"geneSet":"WP35","enrichmentRatio":5.4182288299935362,"FDR":1,"overlap":1},{"geneSet":"WP2355","enrichmentRatio":5.3017077798861481,"FDR":1,"overlap":1},{"geneSet":"WP2806","enrichmentRatio":5.1900928792569658,"FDR":1,"overlap":1},{"geneSet":"WP4258","enrichmentRatio":5.0312124849939979,"FDR":1,"overlap":1},{"geneSet":"WP5090","enrichmentRatio":4.6080263881253432,"FDR":1,"overlap":1},{"geneSet":"WP428","enrichmentRatio":4.3250773993808043,"FDR":1,"overlap":1},{"geneSet":"WP236","enrichmentRatio":3.7638078132016166,"FDR":1,"overlap":1},{"geneSet":"WP4262","enrichmentRatio":3.1810246679316889,"FDR":1,"overlap":1},{"geneSet":"WP481","enrichmentRatio":3.0816176470588235,"FDR":1,"overlap":1},{"geneSet":"WP4239","enrichmentRatio":3.0435729847494555,"FDR":1,"overlap":1},{"geneSet":"WP2877","enrichmentRatio":2.6366782006920415,"FDR":1,"overlap":1},{"geneSet":"WP306","enrichmentRatio":2.4776825302985519,"FDR":1,"overlap":1},{"geneSet":"WP5115","enrichmentRatio":2.2617377226119806,"FDR":1,"overlap":1},{"geneSet":"WP455","enrichmentRatio":1.8891142663962135,"FDR":1,"overlap":1},{"geneSet":"WP5124","enrichmentRatio":1.8747483784388281,"FDR":1,"overlap":1},{"geneSet":"WP2059","enrichmentRatio":1.8397717295873572,"FDR":1,"overlap":1},{"geneSet":"WP2882","enrichmentRatio":1.5702510303484452,"FDR":1,"overlap":1},{"geneSet":"WP3888","enrichmentRatio":1.1413398692810457,"FDR":1,"overlap":1},{"geneSet":"WP5087","enrichmentRatio":1.1231408280852204,"FDR":1,"overlap":1}];
var geneTables = {"WP80":[{"userId":"P2RY6","geneSymbol":"P2RY6","geneName":"pyrimidinergic receptor P2Y6","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=5031","entrezgene":"5031"}],"WP4584":[{"userId":"UPB1","geneSymbol":"UPB1","geneName":"beta-ureidopropionase 1","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=51733","entrezgene":"51733"}],"WP3664":[{"userId":"SFRP4","geneSymbol":"SFRP4","geneName":"secreted frizzled related protein 4","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=6424","entrezgene":"6424"}],"WP5172":[{"userId":"CP","geneSymbol":"CP","geneName":"ceruloplasmin","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=1356","entrezgene":"1356"}],"WP4225":[{"userId":"UPB1","geneSymbol":"UPB1","geneName":"beta-ureidopropionase 1","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=51733","entrezgene":"51733"}],"WP4900":[{"userId":"P2RY6","geneSymbol":"P2RY6","geneName":"pyrimidinergic receptor P2Y6","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=5031","entrezgene":"5031"}],"WP1601":[{"userId":"UPB1","geneSymbol":"UPB1","geneName":"beta-ureidopropionase 1","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=51733","entrezgene":"51733"}],"WP500":[{"userId":"PYGM","geneSymbol":"PYGM","geneName":"glycogen phosphorylase, muscle associated","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=5837","entrezgene":"5837"}],"WP716":[{"userId":"DHRS3","geneSymbol":"DHRS3","geneName":"dehydrogenase/reductase 3","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=9249","entrezgene":"9249"}],"WP4577":[{"userId":"CP","geneSymbol":"CP","geneName":"ceruloplasmin","gLink":"https://www.ncbi.nlm.nih.gov/gene/?term=1356","entrezgene":"1356"}]};
var clusters = {"wsc":{"representatives":["WP80","WP4584","WP3664","WP5172","WP500","WP716"],"coverage":[1]}};

wgLib({
	enrichDatabase: "pathway_WikiPathways",
	organism: "hsapiens",
	hostName: "https://2024.webgestalt.org",
	enrichment: enrichment,
	background: background,
	geneTables: geneTables,
	clusters: clusters,
	enrichMethod: "ora",
	mappedInputSize: 17,
	standardId: "entrezgene",
	volcano: {
		labelTypes: ["geneSet", "description"],
		dimensions: { width: 960, height: 500 },
		autoPlaceLabels: true,
		y: "FDR",

		x: "enrichmentRatio",
		xLabel: "log2 of enrichment ratio",
		radius: "overlap",
		color: "overlap",
			tooltipContent: ["geneSet", "description", "enrichmentRatio", "FDR", "pValue", "overlap"]
	},
	bar: {
		dimensions: { width: 960, height: 300 },
		labelWidth: 200,
		tooltipContent: ["geneSet", "description", "enrichmentRatio", "FDR", "pValue", "overlap"],
		valueKey: "enrichmentRatio",
		xLabel: "Enrichment ratio"
	}
})


</script>
<footer class="footer has-text-centered">
<p>WebGestalt is currently developed and maintained by Yuxing Liao, Suhas Vasaikar, Zhiao Shi and Bing Zhang at the <a target="_new" href=http://www.zhang-lab.org> Zhang Lab</a>. Other people who have made significant contribution to the project include Jing Wang, Dexter Duncan, Stefan Kirov and Jay Snoddy.</p>
<p><b>Funding credits:</b> NIH/NCI (U24 CA210954); Leidos (15X038); CPRIT (RR160027); NIH/NIAAA (U01 AA016662, U01 AA013512); NIH/NIDA (P01 DA015027); NIH/NIMH (P50 MH078028, P50 MH096972); NIH/NCI (U24 CA159988); NIH/NIGMS (R01 GM088822).
</p>
</footer></body>
</html>